<img src='Image/logo.png' width='150'>


# Bioinformatics approaches for polyphenolic compounds production in _Escherichia coli_

### **Introduction**

Prenylflavonoids and furanocoumarins, among the phenolic compounds, have been regarded as appealing metabolites to produce due to their wide range of applications, particularly in medicine. These compounds are difficult and expensive to chemically synthesize and purify. For that reason, they are widely extracted from a variety of plants, despite their low concentration. Therefore, improving the production of these compounds is crucial. In this work, an _Escherichia coli_ strain was designed to produce these polyphenolic compounds using known heterologous pathways. For that, genome-scale metabolic models were used to simulate in vivo behavior and optimize the mutant strain using evolutionary algorithms. It was possible to find strategies that could produce these compounds. However, the estimated flux values were very low, and some approaches use a significant number of genetic modifications which could be a problem in a laboratory context. 

# Goals
This project aims to design an _E. coli_ strain able to produce polyphenolic compounds using known heterologous pathways. The first step consisted in the identification of the best candidate genes for the design of the polyphenolic pathways. Then, the _in silico_ modeling of the pathways and insertion into available metabolic model reconstructions of _E. coli_ was performed. Finally, the mutant strain was optimized through a rational design approach.


