{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "e73546d1-336b-489f-8a5e-5e2376486a47",
   "metadata": {},
   "source": [
    "## Informations about the genes obtained in the optimizations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "4ca195e1-4586-433a-8476-831be9c40d48",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import ExPASy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "c1fb068b-4de9-433e-8940-3234d47a8c7b",
   "metadata": {},
   "outputs": [],
   "source": [
    "def file (id_gene ,filename):\n",
    "    '''\n",
    "    A função file utiliza o package Biopython ExPASy para ir buscar ficheiros à Uniprot.\n",
    "    Para a função é dado o id do gene a procurar na base de dados e o nome do ficheiro xml a criar \n",
    "    com os resultados obtidos. \n",
    "    '''\n",
    "    for i in id_gene:\n",
    "        handle = ExPASy.get_sprot_raw(i)\n",
    "        gene = handle.read()\n",
    "        handle.close ()\n",
    "        with open(filename, \"a\") as out_handle:\n",
    "            out_handle.write(gene)\n",
    "    return gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "acff177c-e4ab-472b-a6a0-0c05f07cee3d",
   "metadata": {},
   "outputs": [],
   "source": [
    "def informations (filename):\n",
    "    '''\n",
    "    Para a função informations é dado o nome do ficheiro xml obtido com os resultados da Uniprot e \n",
    "    imprime o ID com o status reviwed e o comprimento da sequência, o nome do gene, source, mutações, \n",
    "    funções, entre outras informções.\n",
    "    '''\n",
    "    with open(filename) as file: \n",
    "        file_list = file.readlines()\n",
    "    for i in range(len(file_list)):\n",
    "        if \"ID\" in file_list[i][:5]: \n",
    "            print('---------------------------')\n",
    "            print()\n",
    "            print(\"Nome de acesso, Status e comprimento da sequência:\",file_list[i][5:].replace(\"    \",\"\"),end=\"\")\n",
    "        if \"AC   \" in file_list[i][:5]: \n",
    "            print(\"ID:\",file_list[i][5:],end=\"\")\n",
    "        if \"DE   \" in file_list[i][:5]: \n",
    "            print(file_list[i][5:],end=\"\")\n",
    "        if \"GN\" in file_list[i][:5]: \n",
    "            print(file_list[i][5:],end=\"\")\n",
    "        if \"OS\" in file_list[i][:5]: \n",
    "            print(\"Source:\",file_list[i][5:],end=\"\")\n",
    "        if \"SUBCELLULAR LOCATION:\" in file_list[i][:30]: \n",
    "            print(\"Localização subcelular:\",file_list[i][30:],end=\"\")\n",
    "        if \"DR\" in file_list[i][:5] and \"PROSITE\" in file_list[i][:15]:\n",
    "            print(\"Domínios PROSITE:\",file_list[i][5:],end=\"\")\n",
    "        if \"RP\" in file_list [i][:5] and 'FUNCTION' in file_list [i]:\n",
    "            print(file_list[i][5:], end=\"\")\n",
    "        if \"CC   \"  in file_list [i][:5] and not \"CC   --\" in file_list [i] and not  \"CC   Co\" in file_list [i] and not  \"CC   Di\" in file_list [i]: \n",
    "            print (file_list [i][5:], end=\"\")\n",
    "    f = open(filename, \"r\")\n",
    "    content = f.read()\n",
    "    return None"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f1ad96fb-97ba-418a-9510-3948090b7fd2",
   "metadata": {},
   "source": [
    "### Informations about xanthotoxin genes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "2aa0cde5-aae0-4138-bd4b-3da3cad3d48c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: BIOA_ECOLI  Reviewed; 429 AA.\n",
      "ID: P12995; Q2MBJ5;\n",
      "RecName: Full=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase;\n",
      "         EC=2.6.1.62 {ECO:0000269|PubMed:1092681, ECO:0000269|PubMed:1092682};\n",
      "AltName: Full=7,8-diamino-pelargonic acid aminotransferase {ECO:0000303|PubMed:1092681};\n",
      "         Short=DAPA AT;\n",
      "         Short=DAPA aminotransferase {ECO:0000303|PubMed:1092682};\n",
      "AltName: Full=7,8-diaminononanoate synthase;\n",
      "         Short=DANS;\n",
      "AltName: Full=Diaminopelargonic acid synthase;\n",
      "Name=bioA; OrderedLocusNames=b0774, JW0757;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, SUBUNIT, SUBSTRATE SPECIFICITY, COFACTOR, AND CATALYTIC ACTIVITY.\n",
      "BIOPHYSICOCHEMICAL PROPERTIES, ACTIVITY REGULATION, FUNCTION, AND CATALYTIC\n",
      "FUNCTION, AND DISRUPTION PHENOTYPE.\n",
      "-!- FUNCTION: Catalyzes the transfer of the alpha-amino group from S-\n",
      "    adenosyl-L-methionine (SAM) to 7-keto-8-aminopelargonic acid (KAPA) to\n",
      "    form 7,8-diaminopelargonic acid (DAPA) (PubMed:1092681,\n",
      "    PubMed:1092682). It is the only animotransferase known to utilize SAM\n",
      "    as an amino donor (PubMed:1092681, PubMed:1092682). Complements a bioU\n",
      "    deletion in Synechocystis PCC 6803 (PubMed:32042199).\n",
      "    {ECO:0000269|PubMed:1092681, ECO:0000269|PubMed:1092682,\n",
      "    ECO:0000269|PubMed:32042199}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=(S)-8-amino-7-oxononanoate + S-adenosyl-L-methionine =\n",
      "      (7R,8S)-7,8-diammoniononanoate + S-adenosyl-4-methylsulfanyl-2-\n",
      "      oxobutanoate; Xref=Rhea:RHEA:16861, ChEBI:CHEBI:16490,\n",
      "      ChEBI:CHEBI:59789, ChEBI:CHEBI:149468, ChEBI:CHEBI:149469;\n",
      "      EC=2.6.1.62; Evidence={ECO:0000269|PubMed:1092681,\n",
      "      ECO:0000269|PubMed:1092682};\n",
      "-!- COFACTOR:\n",
      "    Name=pyridoxal 5'-phosphate; Xref=ChEBI:CHEBI:597326;\n",
      "      Evidence={ECO:0000269|PubMed:1092681};\n",
      "-!- ACTIVITY REGULATION: Inhibited by amiclenomycin. S-adenosyl-L-(2-\n",
      "    hydroxy-4-methylthio)butyric acid and adenosine are competitive\n",
      "    inhibitors with SAM and uncompetitive inhibitors with KAPA as\n",
      "    substrates (PubMed:1092682). S-adenosyl-L-ethionine, adenine and 8-\n",
      "    keto-7-aminopelargonic acid are non-competitive inhibitors with both\n",
      "    substrates (PubMed:1092682). {ECO:0000269|PubMed:1092682,\n",
      "    ECO:0000269|PubMed:12218056, ECO:0000269|PubMed:16042602}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=1.2 uM for KAPA {ECO:0000269|PubMed:1092682,\n",
      "      ECO:0000269|PubMed:12379100};\n",
      "      KM=150 uM for SAM {ECO:0000269|PubMed:1092682,\n",
      "      ECO:0000269|PubMed:12379100};\n",
      "      KM=21 uM for pyridoxamine phosphate (PMP)\n",
      "      {ECO:0000269|PubMed:1092682, ECO:0000269|PubMed:12379100};\n",
      "      KM=32 uM for pyridoxal phosphate (PLP) {ECO:0000269|PubMed:1092682,\n",
      "      ECO:0000269|PubMed:12379100};\n",
      "      KM=1000 uM for 8-keto-7-aminopelargonic acid\n",
      "      {ECO:0000269|PubMed:1092682, ECO:0000269|PubMed:12379100};\n",
      "      Vmax=0.16 umol/min/mg enzyme with KAPA as substrate\n",
      "      {ECO:0000269|PubMed:1092682, ECO:0000269|PubMed:12379100};\n",
      "      Vmax=0.027 umol/min/mg enzyme with 8-keto-7-aminopelargonic acid as\n",
      "      substrate {ECO:0000269|PubMed:1092682, ECO:0000269|PubMed:12379100};\n",
      "-!- PATHWAY: Cofactor biosynthesis; biotin biosynthesis; 7,8-\n",
      "    diaminononanoate from 8-amino-7-oxononanoate (SAM route): step 1/1.\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000269|PubMed:10452893,\n",
      "    ECO:0000269|PubMed:1092681, ECO:0000269|PubMed:12218056,\n",
      "    ECO:0000269|PubMed:12379100, ECO:0000269|PubMed:14756557}.\n",
      "Localização subcelular:  Cytoplasm {ECO:0000250}.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.\n",
      "-!- INDUCTION: Repressed by BirA. {ECO:0000269|PubMed:6456358}.\n",
      "-!- DISRUPTION PHENOTYPE: Loss of biotin synthesis.\n",
      "    {ECO:0000269|PubMed:32042199}.\n",
      "-!- MISCELLANEOUS: Catalysis proceeds by a classical ping-pong bi-bi\n",
      "    reaction mechanism.\n",
      "-!- SIMILARITY: Belongs to the class-III pyridoxal-phosphate-dependent\n",
      "    aminotransferase family. BioA subfamily. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00600; AA_TRANSFER_CLASS_3; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MOBA_ECOLI  Reviewed; 194 AA.\n",
      "ID: P32173; Q2M8F5; Q9LBV0;\n",
      "RecName: Full=Molybdenum cofactor guanylyltransferase;\n",
      "         Short=MoCo guanylyltransferase;\n",
      "         EC=2.7.7.77;\n",
      "AltName: Full=GTP:molybdopterin guanylyltransferase;\n",
      "AltName: Full=Mo-MPT guanylyltransferase;\n",
      "AltName: Full=Molybdopterin guanylyltransferase;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide biosynthesis protein A;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide synthase;\n",
      "         Short=MGD synthase;\n",
      "AltName: Full=Protein FA;\n",
      "Name=mobA; Synonyms=chlB, mob, narB; OrderedLocusNames=b3857, JW3829;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 1-5, FUNCTION IN MGD BIOSYNTHESIS, AND SUBUNIT.\n",
      "FUNCTION IN MGD BIOSYNTHESIS, AND DISRUPTION PHENOTYPE.\n",
      "FUNCTION IN BIS(MGD) AND MGD BIOSYNTHESIS, CATALYTIC ACTIVITY, SUBSTRATE\n",
      "FUNCTION, CATALYTIC ACTIVITY, KINETIC PARAMETERS, DOMAIN, AND MUTAGENESIS\n",
      "-!- FUNCTION: Transfers a GMP moiety from GTP to Mo-molybdopterin (Mo-MPT)\n",
      "    cofactor (Moco or molybdenum cofactor) to form Mo-molybdopterin guanine\n",
      "    dinucleotide (Mo-MGD) cofactor. Is also involved in the biosynthesis of\n",
      "    the bis-MGD form of the Moco cofactor (Mo-bisMGD) in which the metal is\n",
      "    symmetrically ligated by the dithiolene groups of two MGD molecules. Is\n",
      "    necessary and sufficient for the in vitro activation of the DMSOR\n",
      "    molybdoenzyme that uses the Mo-bisMGD form of molybdenum cofactor,\n",
      "    which implies formation and efficient insertion of the cofactor into\n",
      "    the enzyme without the need of a chaperone. Is specific for GTP since\n",
      "    other nucleotides such as ATP and GMP cannot be utilized.\n",
      "    {ECO:0000269|PubMed:10978348, ECO:0000269|PubMed:1648082,\n",
      "    ECO:0000269|PubMed:21081498, ECO:0000269|PubMed:8020507}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=GTP + H(+) + Mo-molybdopterin = diphosphate + Mo-molybdopterin\n",
      "      guanine dinucleotide; Xref=Rhea:RHEA:34243, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:37565, ChEBI:CHEBI:71302,\n",
      "      ChEBI:CHEBI:71310; EC=2.7.7.77;\n",
      "      Evidence={ECO:0000269|PubMed:10978348, ECO:0000269|PubMed:21081498};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:10978347, ECO:0000269|PubMed:10978348};\n",
      "    Name=Mn(2+); Xref=ChEBI:CHEBI:29035;\n",
      "      Evidence={ECO:0000269|PubMed:10978347, ECO:0000269|PubMed:10978348};\n",
      "    Note=Both divalent cations appear to be equally efficient in an vitro\n",
      "    reconstitution assay. {ECO:0000269|PubMed:10978347,\n",
      "    ECO:0000269|PubMed:10978348};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=6.5 uM for GTP {ECO:0000269|PubMed:21081498};\n",
      "-!- SUBUNIT: Monomer. An equilibrium exists between a monomeric and\n",
      "    oligomeric form of the enzyme, which could be an octamer; whether this\n",
      "    oligomeric arrangement is of functional relevance is unclear. Interacts\n",
      "    with MoeA and MobB in vivo. {ECO:0000269|PubMed:10978347,\n",
      "    ECO:0000269|PubMed:12372836, ECO:0000269|PubMed:8020507}.\n",
      "-!- INTERACTION:\n",
      "    P32173; P12281: moeA; NbExp=3; IntAct=EBI-1133881, EBI-554393;\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- INDUCTION: Is expressed at very low levels under both aerobic and\n",
      "    anaerobic growth conditions. {ECO:0000269|PubMed:7551035}.\n",
      "-!- DOMAIN: The N-terminal domain determines nucleotide recognition and\n",
      "    specific binding, while the C-terminal domain determines the specific\n",
      "    binding to the target protein. When the N-terminal domain of MobA is\n",
      "    fused to the C-terminal domain of MocA, comparable kinetic constants as\n",
      "    wild-type MobA are obtained with GTP, and the activity with CTP is\n",
      "    completely lost. Consistent results are obtained when the N-terminal\n",
      "    domain of MocA is fused to the C-terminal domain of MobA: the kinetic\n",
      "    constants with CTP are comparable with the ones found for wild-type\n",
      "    MocA, although no activity with GTP is detected.\n",
      "    {ECO:0000269|PubMed:21081498}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene are chlorate-resistant,\n",
      "    fail to synthesize MGD and accumulate elevated quantities of MPT.\n",
      "    {ECO:0000269|PubMed:1648082}.\n",
      "-!- SIMILARITY: Belongs to the MobA family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: NADC_ECOLI  Reviewed; 297 AA.\n",
      "ID: P30011; Q2MCG5;\n",
      "RecName: Full=Nicotinate-nucleotide pyrophosphorylase [carboxylating];\n",
      "         EC=2.4.2.19 {ECO:0000269|PubMed:8561507};\n",
      "AltName: Full=Quinolinate phosphoribosyltransferase [decarboxylating] {ECO:0000303|PubMed:8561507};\n",
      "         Short=QAPRTase;\n",
      "Name=nadC; OrderedLocusNames=b0109, JW0105;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, CATALYTIC ACTIVITY,\n",
      "-!- FUNCTION: Involved in the catabolism of quinolinic acid (QA).\n",
      "    {ECO:0000269|PubMed:8561507}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=CO2 + diphosphate + nicotinate beta-D-ribonucleotide = 5-\n",
      "      phospho-alpha-D-ribose 1-diphosphate + 2 H(+) + quinolinate;\n",
      "      Xref=Rhea:RHEA:12733, ChEBI:CHEBI:15378, ChEBI:CHEBI:16526,\n",
      "      ChEBI:CHEBI:29959, ChEBI:CHEBI:33019, ChEBI:CHEBI:57502,\n",
      "      ChEBI:CHEBI:58017; EC=2.4.2.19;\n",
      "      Evidence={ECO:0000269|PubMed:8561507};\n",
      "    PhysiologicalDirection=right-to-left; Xref=Rhea:RHEA:12735;\n",
      "      Evidence={ECO:0000269|PubMed:8561507};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=15.6 uM for 5-phospho-alpha-D-ribose (at pH 7.8)\n",
      "      {ECO:0000269|PubMed:8561507};\n",
      "      KM=6.4 uM for quinolinate (at pH 7.8) {ECO:0000269|PubMed:8561507};\n",
      "      Vmax=0.96 umol/min/mg enzyme (at pH 7.8)\n",
      "      {ECO:0000269|PubMed:8561507};\n",
      "-!- PATHWAY: Cofactor biosynthesis; NAD(+) biosynthesis; nicotinate D-\n",
      "    ribonucleotide from quinolinate: step 1/1.\n",
      "    {ECO:0000305|PubMed:8561507}.\n",
      "-!- SUBUNIT: Homodimer Hexamer formed by 3 homodimers (By similarity).\n",
      "    Homodimer. {ECO:0000250}.\n",
      "-!- SIMILARITY: Belongs to the NadC/ModD family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MOBB_ECOLI  Reviewed; 175 AA.\n",
      "ID: P32125; P76770; Q2M8F4;\n",
      "RecName: Full=Molybdopterin-guanine dinucleotide biosynthesis adapter protein;\n",
      "         Short=MGD biosynthesis adapter protein;\n",
      "AltName: Full=Molybdenum cofactor biosynthesis adapter protein;\n",
      "         Short=Moco biosynthesis adapter protein;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide biosynthesis protein B;\n",
      "Name=mobB; Synonyms=yihC; OrderedLocusNames=b3856, JW5575;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 2-13, GTP-BINDING, GTPASE ACTIVITY, FUNCTION, AND\n",
      "X-RAY CRYSTALLOGRAPHY (1.90 ANGSTROMS) IN COMPLEX WITH SULFATE, FUNCTION,\n",
      "-!- FUNCTION: GTP-binding protein that is not required for the biosynthesis\n",
      "    of Mo-molybdopterin guanine dinucleotide (Mo-MGD) cofactor, and not\n",
      "    necessary for the formation of active molybdoenzymes using this form of\n",
      "    molybdenum cofactor. May act as an adapter protein to achieve the\n",
      "    efficient biosynthesis and utilization of MGD. Displays a weak\n",
      "    intrinsic GTPase activity. Is also able to bind the nucleotides ATP,\n",
      "    TTP and GDP, but with lower affinity than GTP.\n",
      "    {ECO:0000269|PubMed:12682065, ECO:0000269|PubMed:9219527}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=7.5 uM for GTP;\n",
      "      Note=kcat is 0.003 min(-1) for the GTPase activity.;\n",
      "-!- SUBUNIT: Homodimer. Interacts with MobA, MogA and MoeA in vivo.\n",
      "    {ECO:0000269|PubMed:12372836, ECO:0000269|PubMed:12682065,\n",
      "    ECO:0000269|PubMed:14646116, ECO:0000269|PubMed:9219527}.\n",
      "-!- INTERACTION:\n",
      "    P32125; P30749: moaE; NbExp=2; IntAct=EBI-879965, EBI-554376;\n",
      "-!- INDUCTION: Is expressed at very low levels under both aerobic and\n",
      "    anaerobic growth conditions. {ECO:0000269|PubMed:7551035}.\n",
      "-!- SIMILARITY: Belongs to the MobB family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAB02991.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "    Sequence=BAE77452.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: NORV_ECOLI  Reviewed; 479 AA.\n",
      "ID: Q46877; Q2MAB9; Q46708; Q46709;\n",
      "RecName: Full=Anaerobic nitric oxide reductase flavorubredoxin;\n",
      "         Short=FlRd;\n",
      "         Short=FlavoRb;\n",
      "Name=norV; Synonyms=flrD, ygaI, ygaJ, ygaK;\n",
      "OrderedLocusNames=b2710, JW2680;\n",
      "Source: Escherichia coli (strain K12).\n",
      "IDENTIFICATION OF FUNCTION.\n",
      "IDENTIFICATION OF FUNCTION.\n",
      "-!- FUNCTION: Anaerobic nitric oxide reductase; uses NADH to detoxify\n",
      "    nitric oxide (NO), protecting several 4Fe-4S NO-sensitive enzymes. Has\n",
      "    at least 2 reductase partners, only one of which (NorW, flavorubredoxin\n",
      "    reductase) has been identified. NO probably binds to the di-iron\n",
      "    center; electrons enter from the reductase at rubredoxin and are\n",
      "    transferred sequentially to the FMN center and the di-iron center. Also\n",
      "    able to function as an aerobic oxygen reductase.\n",
      "-!- COFACTOR:\n",
      "    Name=Fe cation; Xref=ChEBI:CHEBI:24875;\n",
      "    Note=Binds 3 Fe cations per monomer.;\n",
      "-!- COFACTOR:\n",
      "    Name=FMN; Xref=ChEBI:CHEBI:58210;\n",
      "    Note=Binds 1 FMN per monomer.;\n",
      "-!- PATHWAY: Nitrogen metabolism; nitric oxide reduction.\n",
      "-!- SUBUNIT: Homotetramer.\n",
      "-!- INTERACTION:\n",
      "    Q46877; Q46877: norV; NbExp=2; IntAct=EBI-557904, EBI-557904;\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- INDUCTION: Submicromolar concentrations of NO induce NorV-dependent NO\n",
      "    consumption; as NO reductase is sensitive to oxygen, no activity is\n",
      "    detected in its presence. Repressed by oxygen in the presence of NO.\n",
      "    Different effects on anaerobic induction in the absence of NO, and in\n",
      "    the presence of N(O)3- or N(O)2- have been reported. Transcription is\n",
      "    negatively regulated by FNR and does not require NarL or NarP.\n",
      "-!- SIMILARITY: In the N-terminal section; belongs to the zinc metallo-\n",
      "    hydrolase group 3 family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=BAA05933.1; Type=Frameshift; Evidence={ECO:0000305};\n",
      "Domínios PROSITE: PROSITE; PS50902; FLAVODOXIN_LIKE; 1.\n",
      "Domínios PROSITE: PROSITE; PS50903; RUBREDOXIN_LIKE; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: NORW_ECOLI  Reviewed; 377 AA.\n",
      "ID: P37596; Q2MAB8;\n",
      "RecName: Full=Nitric oxide reductase FlRd-NAD(+) reductase;\n",
      "         EC=1.18.1.-;\n",
      "AltName: Full=Flavorubredoxin reductase;\n",
      "         Short=FlRd-reductase;\n",
      "         Short=FlavoRb reductase;\n",
      "Name=norW; Synonyms=flrR, ygbD; OrderedLocusNames=b2711, JW2681;\n",
      "Source: Escherichia coli (strain K12).\n",
      "IDENTIFICATION OF FUNCTION.\n",
      "IDENTIFICATION OF FUNCTION.\n",
      "-!- FUNCTION: One of at least two accessory proteins for anaerobic nitric\n",
      "    oxide (NO) reductase. Reduces the rubredoxin moiety of NO reductase.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=H(+) + NAD(+) + 2 reduced [nitric oxide reductase rubredoxin\n",
      "      domain] = NADH + 2 oxidized [nitric oxide reductase rubredoxin\n",
      "      domain]; Xref=Rhea:RHEA:42960, Rhea:RHEA-COMP:10304, Rhea:RHEA-\n",
      "      COMP:10305, ChEBI:CHEBI:15378, ChEBI:CHEBI:29033, ChEBI:CHEBI:29034,\n",
      "      ChEBI:CHEBI:57540, ChEBI:CHEBI:57945;\n",
      "-!- COFACTOR:\n",
      "    Name=FAD; Xref=ChEBI:CHEBI:57692; Evidence={ECO:0000305};\n",
      "-!- PATHWAY: Nitrogen metabolism; nitric oxide reduction.\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- INDUCTION: Submicromolar concentrations of NO induce anaerobic\n",
      "    expression. Repressed by oxygen in the presence of NO.\n",
      "-!- SIMILARITY: Belongs to the FAD-dependent oxidoreductase family.\n",
      "    {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: KGUA_ECOLI  Reviewed; 207 AA.\n",
      "ID: P60546; P24234; Q2M7W1;\n",
      "RecName: Full=Guanylate kinase;\n",
      "         EC=2.7.4.8 {ECO:0000269|PubMed:8390989};\n",
      "AltName: Full=GMP kinase;\n",
      "Name=gmk; Synonyms=spoR; OrderedLocusNames=b3648, JW3623;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, CATALYTIC ACTIVITY, AND\n",
      "-!- FUNCTION: Essential for recycling GMP and indirectly, cGMP.\n",
      "    {ECO:0000269|PubMed:8390989}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + GMP = ADP + GDP; Xref=Rhea:RHEA:20780,\n",
      "      ChEBI:CHEBI:30616, ChEBI:CHEBI:58115, ChEBI:CHEBI:58189,\n",
      "      ChEBI:CHEBI:456216; EC=2.7.4.8;\n",
      "      Evidence={ECO:0000269|PubMed:8390989};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:20781;\n",
      "      Evidence={ECO:0000305|PubMed:8390989};\n",
      "-!- SUBUNIT: Homotetramer (under low ionic conditions) or homodimer (under\n",
      "    high ionic conditions). {ECO:0000269|PubMed:8390989}.\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- SIMILARITY: Belongs to the guanylate kinase family. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00856; GUANYLATE_KINASE_1; 1.\n",
      "Domínios PROSITE: PROSITE; PS50052; GUANYLATE_KINASE_2; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MALE_ECOLI  Reviewed; 396 AA.\n",
      "ID: P0AEX9; P02928; Q2M6S0;\n",
      "RecName: Full=Maltose/maltodextrin-binding periplasmic protein {ECO:0000305};\n",
      "AltName: Full=MMBP {ECO:0000305};\n",
      "AltName: Full=Maltodextrin-binding protein {ECO:0000305};\n",
      "AltName: Full=Maltose-binding protein {ECO:0000303|PubMed:4215651};\n",
      "         Short=MBP {ECO:0000303|PubMed:6088507};\n",
      "Flags: Precursor;\n",
      "Name=malE {ECO:0000303|PubMed:4215651}; OrderedLocusNames=b4034, JW3994;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, SUBCELLULAR LOCATION, AND INDUCTION.\n",
      "FUNCTION, AND BIOPHYSICOCHEMICAL PROPERTIES.\n",
      "FUNCTION, AND SUBUNIT.\n",
      "FUNCTION, AND SUBUNIT.\n",
      "-!- FUNCTION: Part of the ABC transporter complex MalEFGK involved in\n",
      "    maltose/maltodextrin import. Binds maltose and higher maltodextrins\n",
      "    such as maltotriose. {ECO:0000269|PubMed:2026607,\n",
      "    ECO:0000269|PubMed:2155217, ECO:0000269|PubMed:4215651,\n",
      "    ECO:0000269|PubMed:776623}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=0.9 uM for maltose {ECO:0000269|PubMed:776623};\n",
      "      KM=2 uM for maltotriose {ECO:0000269|PubMed:776623};\n",
      "-!- SUBUNIT: The complex is composed of two ATP-binding proteins (MalK),\n",
      "    two transmembrane proteins (MalG and MalF) and a solute-binding protein\n",
      "    (MalE). {ECO:0000269|PubMed:2026607, ECO:0000269|PubMed:2155217}.\n",
      "-!- INTERACTION:\n",
      "    P0AEX9; P0A6F5: groEL; NbExp=3; IntAct=EBI-369910, EBI-543750;\n",
      "    P0AEX9; P02916: malF; NbExp=5; IntAct=EBI-369910, EBI-1118919;\n",
      "Localização subcelular:  Periplasm {ECO:0000269|PubMed:4215651}.\n",
      "-!- SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:4215651}.\n",
      "-!- INDUCTION: Induced by maltose and the transcriptional regulator MalT\n",
      "    (PubMed:4215651). Permanently repressed by cold shock in a PNPase-\n",
      "    dependent fashion (PubMed:14527658). {ECO:0000269|PubMed:14527658,\n",
      "    ECO:0000269|PubMed:4215651}.\n",
      "-!- SIMILARITY: Belongs to the bacterial solute-binding protein 1 family.\n",
      "    {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS01037; SBP_BACTERIAL_1; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MALF_ECOLI  Reviewed; 514 AA.\n",
      "ID: P02916; Q2M6S1;\n",
      "RecName: Full=Maltose/maltodextrin transport system permease protein MalF {ECO:0000305};\n",
      "Name=malF {ECO:0000303|PubMed:6088520}; OrderedLocusNames=b4033, JW3993;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, AND SUBUNIT.\n",
      "FUNCTION, AND SUBUNIT.\n",
      "-!- FUNCTION: Part of the ABC transporter complex MalEFGK involved in\n",
      "    maltose/maltodextrin import. Probably responsible for the translocation\n",
      "    of the substrate across the membrane. {ECO:0000269|PubMed:2026607,\n",
      "    ECO:0000269|PubMed:2155217}.\n",
      "-!- SUBUNIT: The complex is composed of two ATP-binding proteins (MalK),\n",
      "    two transmembrane proteins (MalG and MalF) and a solute-binding protein\n",
      "    (MalE). Protein stability and stable complex formation require YidC.\n",
      "    {ECO:0000269|PubMed:10809785, ECO:0000269|PubMed:18456666,\n",
      "    ECO:0000269|PubMed:2026607, ECO:0000269|PubMed:2155217}.\n",
      "-!- INTERACTION:\n",
      "    P02916; P0AEX9: malE; NbExp=5; IntAct=EBI-1118919, EBI-369910;\n",
      "    P02916; P68183: malG; NbExp=5; IntAct=EBI-1118919, EBI-6400985;\n",
      "Localização subcelular:  Cell inner membrane\n",
      "-!- SUBCELLULAR LOCATION: Cell inner membrane\n",
      "    {ECO:0000269|PubMed:18456666}; Multi-pass membrane protein\n",
      "    {ECO:0000255|PROSITE-ProRule:PRU00441, ECO:0000269|PubMed:18456666};\n",
      "    Periplasmic side {ECO:0000269|PubMed:18456666}. Note=A substantial\n",
      "    portion of it protrudes into the periplasmic space; inserts in an\n",
      "    SRP- and Sec-dependent, YidC-independent fashion into the membrane.\n",
      "-!- MISCELLANEOUS: When MalF EAA loop mutations are made concomitantly with\n",
      "    MalG EAA loop mutations, a complete loss of transport and complex\n",
      "    formation is observed, except for the Gly-407. This suggests that the\n",
      "    MalF-MalG interaction may be important for the proper assembly and also\n",
      "    for the correct function of the transporter.\n",
      "-!- SIMILARITY: Belongs to the binding-protein-dependent transport system\n",
      "    permease family. MalFG subfamily. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS50928; ABC_TM1; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MALG_ECOLI  Reviewed; 296 AA.\n",
      "ID: P68183; P07622; Q2M6S2;\n",
      "RecName: Full=Maltose/maltodextrin transport system permease protein MalG {ECO:0000305};\n",
      "Name=malG {ECO:0000303|PubMed:3000770}; OrderedLocusNames=b4032, JW3992;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, AND SUBUNIT.\n",
      "FUNCTION, AND SUBUNIT.\n",
      "-!- FUNCTION: Part of the ABC transporter complex MalEFGK involved in\n",
      "    maltose/maltodextrin import. Probably responsible for the translocation\n",
      "    of the substrate across the membrane. {ECO:0000269|PubMed:2026607,\n",
      "    ECO:0000269|PubMed:2155217}.\n",
      "-!- SUBUNIT: The complex is composed of two ATP-binding proteins (MalK),\n",
      "    two transmembrane proteins (MalG and MalF) and a solute-binding protein\n",
      "    (MalE). Protein stability and stable complex formation require YidC.\n",
      "    {ECO:0000269|PubMed:10809785, ECO:0000269|PubMed:18456666,\n",
      "    ECO:0000269|PubMed:2026607, ECO:0000269|PubMed:2155217}.\n",
      "-!- INTERACTION:\n",
      "    P68183; P02916: malF; NbExp=5; IntAct=EBI-6400985, EBI-1118919;\n",
      "Localização subcelular:  Cell inner membrane {ECO:0000269|PubMed:18456666,\n",
      "-!- SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000269|PubMed:18456666,\n",
      "    ECO:0000269|PubMed:2233678}; Multi-pass membrane protein\n",
      "    {ECO:0000255|PROSITE-ProRule:PRU00441, ECO:0000269|PubMed:18456666,\n",
      "    ECO:0000269|PubMed:2233678}.\n",
      "-!- MISCELLANEOUS: When MalF EAA loop mutations are made concomitantly with\n",
      "    MalG EAA loop mutations, a complete loss of transport and complex\n",
      "    formation is observed. This suggests that the MalF-MalG interaction may\n",
      "    be important for the proper assembly and also for the correct function\n",
      "    of the transporter.\n",
      "-!- SIMILARITY: Belongs to the binding-protein-dependent transport system\n",
      "    permease family. MalFG subfamily. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS50928; ABC_TM1; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MALK_ECOLI  Reviewed; 371 AA.\n",
      "ID: P68187; P02914; Q2M6R9; Q47348;\n",
      "RecName: Full=Maltose/maltodextrin import ATP-binding protein MalK {ECO:0000255|HAMAP-Rule:MF_01709};\n",
      "         EC=7.5.2.1 {ECO:0000255|HAMAP-Rule:MF_01709, ECO:0000269|PubMed:2026607, ECO:0000269|PubMed:2155217};\n",
      "Name=malK {ECO:0000255|HAMAP-Rule:MF_01709, ECO:0000303|PubMed:6296778};\n",
      "OrderedLocusNames=b4035, JW3995;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND SUBUNIT.\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND SUBUNIT.\n",
      "-!- FUNCTION: Part of the ABC transporter complex MalEFGK involved in\n",
      "    maltose/maltodextrin import. Responsible for energy coupling to the\n",
      "    transport system. {ECO:0000269|PubMed:2026607,\n",
      "    ECO:0000269|PubMed:2155217}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + D-maltose(out) + H2O = ADP + D-maltose(in) + H(+) +\n",
      "      phosphate; Xref=Rhea:RHEA:22132, ChEBI:CHEBI:15377,\n",
      "      ChEBI:CHEBI:15378, ChEBI:CHEBI:17306, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:43474, ChEBI:CHEBI:456216; EC=7.5.2.1;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_01709,\n",
      "      ECO:0000269|PubMed:2026607, ECO:0000269|PubMed:2155217};\n",
      "-!- SUBUNIT: The complex is composed of two ATP-binding proteins (MalK),\n",
      "    two transmembrane proteins (MalG and MalF) and a solute-binding protein\n",
      "    (MalE). Protein stability and stable complex formation require YidC.\n",
      "    {ECO:0000269|PubMed:18456666, ECO:0000269|PubMed:2026607,\n",
      "    ECO:0000269|PubMed:2155217}.\n",
      "Localização subcelular:  Cell inner membrane {ECO:0000255|HAMAP-\n",
      "-!- SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-\n",
      "    Rule:MF_01709, ECO:0000269|PubMed:16079137,\n",
      "    ECO:0000269|PubMed:18456666}; Peripheral membrane protein\n",
      "    {ECO:0000255|HAMAP-Rule:MF_01709, ECO:0000269|PubMed:16079137,\n",
      "    ECO:0000269|PubMed:18456666}.\n",
      "-!- MISCELLANEOUS: Target for inducer exclusion, mediated by the\n",
      "    unphosphorylated enzyme III of the phosphotransferase system for\n",
      "    glucose and resulting in the inhibition of maltose transport.\n",
      "    {ECO:0000269|PubMed:2250006}.\n",
      "-!- MISCELLANEOUS: Acts as a repressor of mal genes. In absence of maltose,\n",
      "    the C-terminus of MalK interacts with MalT, then MalT becomes inactive\n",
      "    and the mal genes are not activated. In presence of maltose, MalK is\n",
      "    tightly associated with the MalFG complex and has no affinity for MalT.\n",
      "    {ECO:0000269|PubMed:9822819}.\n",
      "-!- SIMILARITY: Belongs to the ABC transporter superfamily.\n",
      "    Maltooligosaccharide importer (TC 3.A.1.1.1) family.\n",
      "    {ECO:0000255|HAMAP-Rule:MF_01709}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=CAA23582.1; Type=Erroneous initiation; Note=Extended N-terminus.; Evidence={ECO:0000305};\n",
      "Domínios PROSITE: PROSITE; PS00211; ABC_TRANSPORTER_1; 1.\n",
      "Domínios PROSITE: PROSITE; PS50893; ABC_TRANSPORTER_2; 1.\n",
      "Domínios PROSITE: PROSITE; PS51245; MALK; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: LCFA_ECOLI  Reviewed; 561 AA.\n",
      "ID: P69451; P29212;\n",
      "RecName: Full=Long-chain-fatty-acid--CoA ligase;\n",
      "         EC=6.2.1.3 {ECO:0000269|PubMed:15213221};\n",
      "AltName: Full=Long-chain acyl-CoA synthetase;\n",
      "         Short=Acyl-CoA synthetase;\n",
      "Name=fadD; Synonyms=oldD; OrderedLocusNames=b1805, JW1794;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION IN AEROBIC BETA-OXIDATION PATHWAY, DISRUPTION PHENOTYPE, AND\n",
      "FUNCTION, CATALYTIC ACTIVITY, COFACTOR, SUBSTRATE SPECIFICITY,\n",
      "-!- FUNCTION: Catalyzes the esterification, concomitant with transport, of\n",
      "    exogenous long-chain fatty acids into metabolically active CoA\n",
      "    thioesters for subsequent degradation or incorporation into\n",
      "    phospholipids. Activity is the highest with fatty acid substrates of >\n",
      "    10 carbon atoms (PubMed:15213221). Is involved in the aerobic beta-\n",
      "    oxidative degradation of fatty acids, which allows aerobic growth of\n",
      "    E.coli on fatty acids as a sole carbon and energy source\n",
      "    (PubMed:12535077). {ECO:0000269|PubMed:12535077,\n",
      "    ECO:0000269|PubMed:15213221}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a long-chain fatty acid + ATP + CoA = a long-chain fatty acyl-\n",
      "      CoA + AMP + diphosphate; Xref=Rhea:RHEA:15421, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57560,\n",
      "      ChEBI:CHEBI:83139, ChEBI:CHEBI:456215; EC=6.2.1.3;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:15422;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + hexadecanoate = AMP + diphosphate + hexadecanoyl-\n",
      "      CoA; Xref=Rhea:RHEA:30751, ChEBI:CHEBI:7896, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57379,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:30752;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + dodecanoate = AMP + diphosphate + dodecanoyl-CoA;\n",
      "      Xref=Rhea:RHEA:33623, ChEBI:CHEBI:18262, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57375,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:33624;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=(9Z)-octadecenoate + ATP + CoA = (9Z)-octadecenoyl-CoA + AMP +\n",
      "      diphosphate; Xref=Rhea:RHEA:33607, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:30823, ChEBI:CHEBI:33019, ChEBI:CHEBI:57287,\n",
      "      ChEBI:CHEBI:57387, ChEBI:CHEBI:456215;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:33608;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + tetradecanoate = AMP + diphosphate +\n",
      "      tetradecanoyl-CoA; Xref=Rhea:RHEA:33619, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:30807, ChEBI:CHEBI:33019, ChEBI:CHEBI:57287,\n",
      "      ChEBI:CHEBI:57385, ChEBI:CHEBI:456215;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:33620;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + decanoate = AMP + decanoyl-CoA + diphosphate;\n",
      "      Xref=Rhea:RHEA:33627, ChEBI:CHEBI:27689, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:61430,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:33628;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + octanoate = AMP + diphosphate + octanoyl-CoA;\n",
      "      Xref=Rhea:RHEA:33631, ChEBI:CHEBI:25646, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57386,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:33632;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    pH dependence:\n",
      "      Optimum pH is 7.5-7.8. {ECO:0000269|PubMed:15213221};\n",
      "-!- PATHWAY: Lipid metabolism; fatty acid beta-oxidation.\n",
      "    {ECO:0000269|PubMed:12535077, ECO:0000305|PubMed:15213221}.\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000250|UniProtKB:P9WQ37}.\n",
      "Localização subcelular:  Cell membrane {ECO:0000250}; Peripheral membrane\n",
      "-!- SUBCELLULAR LOCATION: Cell membrane {ECO:0000250}; Peripheral membrane\n",
      "    protein {ECO:0000250}. Note=Partially membrane-associated.\n",
      "    {ECO:0000250}.\n",
      "-!- INDUCTION: Well expressed during log and stationary phase aerobic\n",
      "    growth but only poorly expressed during anaerobic growth (at protein\n",
      "    level); aerobic expression is suppressed by glucose or C8 fatty acids\n",
      "    (PubMed:15213221). {ECO:0000269|PubMed:15213221}.\n",
      "-!- PTM: Post-translationally cleaved by OmpT (PubMed:15213221).\n",
      "    {ECO:0000269|PubMed:15213221}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene are completely blocked in\n",
      "    aerobic growth on fatty acids. The double fadD fadK deletion mutant\n",
      "    fails to grow on fatty acids under either aerobic or anaerobic\n",
      "    conditions, although fadD mutants grow on fatty acids under anaerobic\n",
      "    conditions. {ECO:0000269|PubMed:12535077}.\n",
      "-!- MISCELLANEOUS: The enzymatic mechanism is a two-step reaction that\n",
      "    proceeds via the intermediate formation of an acyl-adenylate (acyl-AMP)\n",
      "    intermediate. {ECO:0000269|PubMed:15213221}.\n",
      "-!- SIMILARITY: Belongs to the ATP-dependent AMP-binding enzyme family.\n",
      "    {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00455; AMP_BINDING; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: FADK_ECOLI  Reviewed; 566 AA.\n",
      "ID: P38135; P76202; P76902; P76903;\n",
      "RecName: Full=Medium-chain fatty-acid--CoA ligase {ECO:0000305|PubMed:15213221, ECO:0000305|PubMed:19477415};\n",
      "         EC=6.2.1.- {ECO:0000269|PubMed:15213221, ECO:0000269|PubMed:19477415};\n",
      "AltName: Full=Acyl-CoA synthetase {ECO:0000303|PubMed:15213221};\n",
      "         Short=ACS {ECO:0000303|PubMed:15213221};\n",
      "AltName: Full=Fatty acyl-CoA synthetase FadK {ECO:0000305};\n",
      "Name=fadK {ECO:0000303|PubMed:15213221}; Synonyms=ydiD;\n",
      "OrderedLocusNames=b1701, JW5910;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION IN ANAEROBIC BETA-OXIDATION PATHWAY, INDUCTION, DISRUPTION\n",
      "FUNCTION, CATALYTIC ACTIVITY, COFACTOR, SUBSTRATE SPECIFICITY,\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND SUBSTRATE SPECIFICITY.\n",
      "-!- FUNCTION: Catalyzes the esterification, concomitant with transport, of\n",
      "    exogenous fatty acids into metabolically active CoA thioesters for\n",
      "    subsequent degradation or incorporation into phospholipids. Is\n",
      "    maximally active on C6:0, C8:0 and C12:0 fatty acids, while has a low\n",
      "    activity on C14-C18 chain length fatty acids (PubMed:15213221,\n",
      "    PubMed:19477415). Is involved in the anaerobic beta-oxidative\n",
      "    degradation of fatty acids, which allows anaerobic growth of E.coli on\n",
      "    fatty acids as a sole carbon and energy source in the presence of\n",
      "    nitrate or fumarate as a terminal electron acceptor (PubMed:12535077).\n",
      "    Can functionally replace FadD under anaerobic conditions\n",
      "    (PubMed:12535077). {ECO:0000269|PubMed:12535077,\n",
      "    ECO:0000269|PubMed:15213221, ECO:0000269|PubMed:19477415}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + hexanoate = AMP + diphosphate + hexanoyl-CoA;\n",
      "      Xref=Rhea:RHEA:43740, ChEBI:CHEBI:17120, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:62620,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + octanoate = AMP + diphosphate + octanoyl-CoA;\n",
      "      Xref=Rhea:RHEA:33631, ChEBI:CHEBI:25646, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57386,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:15213221,\n",
      "      ECO:0000269|PubMed:19477415};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + CoA + dodecanoate = AMP + diphosphate + dodecanoyl-CoA;\n",
      "      Xref=Rhea:RHEA:33623, ChEBI:CHEBI:18262, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57287, ChEBI:CHEBI:57375,\n",
      "      ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:19477415};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:15213221};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    pH dependence:\n",
      "      Optimum pH is 7.7-8.2. {ECO:0000269|PubMed:15213221};\n",
      "-!- PATHWAY: Lipid metabolism; fatty acid beta-oxidation.\n",
      "    {ECO:0000269|PubMed:12535077, ECO:0000305|PubMed:15213221}.\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000250|UniProtKB:P9WQ37}.\n",
      "Localização subcelular:  Cell membrane; Peripheral membrane protein.\n",
      "-!- SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.\n",
      "    Note=Partially membrane-associated. {ECO:0000305|PubMed:15213221}.\n",
      "-!- INDUCTION: Expression independent of FadR (PubMed:12535077). FadK is\n",
      "    not expressed under aerobic growth conditions, the levels of anaerobic\n",
      "    expression vary with the terminal electron acceptor, with more\n",
      "    expression during growth on fumarate than on nitrate (at protein level)\n",
      "    (PubMed:15213221). {ECO:0000269|PubMed:12535077,\n",
      "    ECO:0000269|PubMed:15213221}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene grow normally on aerobic\n",
      "    oleate plates, but grow relatively poorly on oleate plus nitrate plates\n",
      "    under anaerobic conditions. The double fadD fadK deletion mutant fails\n",
      "    to grow on fatty acids under either aerobic or anaerobic conditions,\n",
      "    although fadD mutants grow on fatty acids under anaerobic conditions.\n",
      "    {ECO:0000269|PubMed:12535077}.\n",
      "-!- MISCELLANEOUS: Probably starts on Met-1; overexpressed protein starting\n",
      "    at Met-19 has lower activity, forms aggregates during reaction and is\n",
      "    unstable in storage at -80 degrees Celsius (PubMed:15213221).\n",
      "    {ECO:0000269|PubMed:15213221}.\n",
      "-!- MISCELLANEOUS: The enzymatic mechanism is a two-step reaction that\n",
      "    proceeds via the intermediate formation of an acyl-adenylate (acyl-AMP)\n",
      "    intermediate. {ECO:0000269|PubMed:15213221}.\n",
      "-!- SIMILARITY: Belongs to the ATP-dependent AMP-binding enzyme family.\n",
      "    {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAC74771.2; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "    Sequence=M69116; Type=Frameshift; Evidence={ECO:0000305};\n",
      "Domínios PROSITE: PROSITE; PS00455; AMP_BINDING; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: ISCU_ECOLI  Reviewed; 128 AA.\n",
      "ID: P0ACD4; P77310;\n",
      "RecName: Full=Iron-sulfur cluster assembly scaffold protein IscU;\n",
      "AltName: Full=Sulfur acceptor protein IscU;\n",
      "Name=iscU; Synonyms=nifU, yfhN; OrderedLocusNames=b2529, JW2513;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION AS A SULFUR ACCEPTOR, INTERACTION WITH ISCS, ACTIVITY REGULATION,\n",
      "FUNCTION, SUBUNIT, AND MUTAGENESIS OF ASP-39; LYS-89; ASN-90; SER-107 AND\n",
      "-!- FUNCTION: A scaffold on which IscS assembles Fe-S clusters. Exists as 2\n",
      "    interconverting forms, a structured (S) and disordered (D) form. The D-\n",
      "    state is the preferred substrate for IscS. Converts to the S-state when\n",
      "    an Fe-S cluster is assembled, which helps it dissociate from IscS to\n",
      "    transfer the Fe-S to an acceptor. It is likely that Fe-S cluster\n",
      "    coordination is flexible as the role of this complex is to build and\n",
      "    then hand off Fe-S clusters. {ECO:0000269|PubMed:11577100,\n",
      "    ECO:0000269|PubMed:22203963}.\n",
      "-!- ACTIVITY REGULATION: Carboxymethylation by iodoacetic acid blocks\n",
      "    sulfur transfer to this protein. {ECO:0000269|PubMed:11577100}.\n",
      "-!- SUBUNIT: Homodimer. Forms a heterotetramer with IscS; each subunit of\n",
      "    the IscS dimer contacts an IscU monomer. {ECO:0000269|PubMed:11577100,\n",
      "    ECO:0000269|PubMed:22203963}.\n",
      "-!- INTERACTION:\n",
      "    P0ACD4; P0A6B7: iscS; NbExp=12; IntAct=EBI-561646, EBI-550055;\n",
      "    P0ACD4; P0ACD4: iscU; NbExp=5; IntAct=EBI-561646, EBI-561646;\n",
      "-!- SIMILARITY: Belongs to the NifU family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: FADE_ECOLI  Reviewed; 814 AA.\n",
      "ID: Q47146; P71283; P75673; Q47680;\n",
      "RecName: Full=Acyl-coenzyme A dehydrogenase {ECO:0000303|PubMed:12057976};\n",
      "         Short=ACDH;\n",
      "         Short=Acyl-CoA dehydrogenase {ECO:0000303|PubMed:12057976};\n",
      "         EC=1.3.8.7 {ECO:0000305|PubMed:12057976};\n",
      "         EC=1.3.8.8 {ECO:0000305|PubMed:12057976};\n",
      "Name=fadE {ECO:0000303|PubMed:12057976}; Synonyms=yafH;\n",
      "OrderedLocusNames=b0221, JW5020;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, PATHWAY, INDUCTION, AND DISRUPTION PHENOTYPE.\n",
      "-!- FUNCTION: Catalyzes the dehydrogenation of acyl-coenzymes A (acyl-CoAs)\n",
      "    to 2-enoyl-CoAs, the first step of the beta-oxidation cycle of fatty\n",
      "    acid degradation. Is required for E.coli to utilize dodecanoate or\n",
      "    oleate as the sole carbon and energy source for growth.\n",
      "    {ECO:0000269|PubMed:12057976}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a medium-chain 2,3-saturated fatty acyl-CoA + H(+) + oxidized\n",
      "      [electron-transfer flavoprotein] = a medium-chain trans-(2E)-enoyl-\n",
      "      CoA + reduced [electron-transfer flavoprotein]; Xref=Rhea:RHEA:14477,\n",
      "      Rhea:RHEA-COMP:10685, Rhea:RHEA-COMP:10686, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:57692, ChEBI:CHEBI:58307, ChEBI:CHEBI:83723,\n",
      "      ChEBI:CHEBI:83726; EC=1.3.8.7;\n",
      "      Evidence={ECO:0000305|PubMed:12057976};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a long-chain 2,3-saturated fatty acyl-CoA + H(+) + oxidized\n",
      "      [electron-transfer flavoprotein] = a long-chain (2E)-enoyl-CoA +\n",
      "      reduced [electron-transfer flavoprotein]; Xref=Rhea:RHEA:17721,\n",
      "      Rhea:RHEA-COMP:10685, Rhea:RHEA-COMP:10686, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:57692, ChEBI:CHEBI:58307, ChEBI:CHEBI:83721,\n",
      "      ChEBI:CHEBI:83727; EC=1.3.8.8;\n",
      "      Evidence={ECO:0000305|PubMed:12057976};\n",
      "-!- COFACTOR:\n",
      "    Name=FAD; Xref=ChEBI:CHEBI:57692;\n",
      "      Evidence={ECO:0000250|UniProtKB:P15651};\n",
      "-!- PATHWAY: Lipid metabolism; fatty acid beta-oxidation.\n",
      "    {ECO:0000269|PubMed:12057976}.\n",
      "-!- INDUCTION: Transcription of fadE is negatively regulated by FadR.\n",
      "    Induced in the presence of fatty acids, in a FadR-dependent manner.\n",
      "    {ECO:0000269|PubMed:12057976}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene are unable to grow on\n",
      "    octanoate, decanoate, dodecanoate, or oleate as the sole carbon and\n",
      "    energy source, but grow well on acetate, while the wild-type strain can\n",
      "    grow on acetate, dodecanoate, and oleate.\n",
      "    {ECO:0000269|PubMed:12057976}.\n",
      "-!- SIMILARITY: Belongs to the acyl-CoA dehydrogenase family.\n",
      "    {ECO:0000305}.\n",
      "-!- CAUTION: Was originally named fadF. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=BAA07583.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n"
     ]
    }
   ],
   "source": [
    "id_gene = ['P12995','P32173','P30011','P32125','Q46877','P37596','P60546','P0AEX9','P02916','P68183','P68187','P69451','P38135','P0ACD4','Q47146']\n",
    "filename = \"xanthotoxin.xml\"\n",
    "file (id_gene,filename)\n",
    "informations (\"xanthotoxin.xml\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e40b44e8-57f4-48c5-b909-6612cb9afcbc",
   "metadata": {},
   "source": [
    "### Informations about xanthohumol genes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6199033d-6414-4e90-9716-33f87c503500",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MOEA_ECOLI  Reviewed; 411 AA.\n",
      "ID: P12281;\n",
      "RecName: Full=Molybdopterin molybdenumtransferase;\n",
      "         Short=MPT Mo-transferase;\n",
      "         EC=2.10.1.1 {ECO:0000269|PubMed:15632135};\n",
      "Name=moeA; Synonyms=bisB, chlE, narE; OrderedLocusNames=b0827, JW0811;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, AND CATALYTIC ACTIVITY.\n",
      "-!- FUNCTION: Catalyzes the insertion of molybdate into adenylated\n",
      "    molybdopterin with the concomitant release of AMP.\n",
      "    {ECO:0000269|PubMed:15632135}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=adenylyl-molybdopterin + H(+) + molybdate = AMP + H2O + Mo-\n",
      "      molybdopterin; Xref=Rhea:RHEA:35047, ChEBI:CHEBI:15377,\n",
      "      ChEBI:CHEBI:15378, ChEBI:CHEBI:36264, ChEBI:CHEBI:62727,\n",
      "      ChEBI:CHEBI:71302, ChEBI:CHEBI:456215; EC=2.10.1.1;\n",
      "      Evidence={ECO:0000269|PubMed:15632135};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:11428898};\n",
      "    Note=Binds 1 Mg(2+) ion per subunit. {ECO:0000269|PubMed:11428898};\n",
      "-!- PATHWAY: Cofactor biosynthesis; molybdopterin biosynthesis.\n",
      "-!- SUBUNIT: Homodimer. Interacts with MobA, MogA and MobB in vivo.\n",
      "    {ECO:0000269|PubMed:11428898, ECO:0000269|PubMed:11525167,\n",
      "    ECO:0000269|PubMed:12372836}.\n",
      "-!- INTERACTION:\n",
      "    P12281; P32173: mobA; NbExp=3; IntAct=EBI-554393, EBI-1133881;\n",
      "    P12281; P12281: moeA; NbExp=5; IntAct=EBI-554393, EBI-554393;\n",
      "    P12281; P0A796: pfkA; NbExp=2; IntAct=EBI-554393, EBI-554405;\n",
      "-!- SIMILARITY: Belongs to the MoeA family. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS01079; MOCF_BIOSYNTHESIS_2; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: SUHB_ECOLI  Reviewed; 267 AA.\n",
      "ID: P0ADG4; P22783; P77511; Q8X2E6;\n",
      "RecName: Full=Nus factor SuhB {ECO:0000303|PubMed:32871103};\n",
      "AltName: Full=Inositol-1-monophosphatase {ECO:0000303|PubMed:8002619};\n",
      "         Short=I-1-Pase {ECO:0000303|PubMed:10747806};\n",
      "         Short=IMPase;\n",
      "         Short=Inositol-1-phosphatase;\n",
      "         EC=3.1.3.25 {ECO:0000269|PubMed:8002619};\n",
      "Name=suhB {ECO:0000303|PubMed:2138605};\n",
      "Synonyms=ssyA {ECO:0000303|PubMed:6389495};\n",
      "OrderedLocusNames=b2533, JW2517;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], AND FUNCTION AS A SUPPRESSOR OF RPOH\n",
      "FUNCTION AS A SUPPRESSOR OF SECY TEMPERATURE-SENSITIVE MUTATION.\n",
      "FUNCTION AS A SUPPRESSOR OF DNAB MISSENSE MUTATION.\n",
      "FUNCTION AS AN INOSITOL MONOPHOSPHATASE, CATALYTIC ACTIVITY, SUBSTRATE\n",
      "FUNCTION AS AN INOSITOL MONOPHOSPHATASE, CATALYTIC ACTIVITY, SUBSTRATE\n",
      "FUNCTION IN RRNA MATURATION, SUBUNIT, AND DISRUPTION PHENOTYPE.\n",
      "FUNCTION IN GENE REGULATION, SUBUNIT, AND INDUCTION.\n",
      "FUNCTION, SUBUNIT, AND COFACTOR.\n",
      "-!- FUNCTION: Part of the processive rRNA transcription and antitermination\n",
      "    complex (rrnTAC). The complex forms an RNA-chaperone ring around the\n",
      "    RNA exit tunnel of RNA polymerase (RNAP). It supports rapid\n",
      "    transcription and antitermination of rRNA operons, cotranscriptional\n",
      "    rRNA folding, and annealing of distal rRNA regions to allow correct\n",
      "    ribosome biogenesis. This subunit may play a central role in organizing\n",
      "    the structure (PubMed:32871103). Involved in 30S ribosomal subunit\n",
      "    biogenesis; thought to be required for loop formation between NusB/NusE\n",
      "    (rpsJ, ribosomal protein S10) bound to boxA upstream of the rRNA\n",
      "    operons and the elongating RNAP complex. This would promote correct co-\n",
      "    transcriptional folding of rRNA. Plays a role in transcription\n",
      "    antitermination in a plasmid context in vivo (PubMed:26980831).\n",
      "    Required for rrn transcription antitermination; required for\n",
      "    integration of NusB/NusE into the antitermination complex\n",
      "    (PubMed:31020314). The Nus factor complex (NusA, NusB, NusE (rpsJ),\n",
      "    NusG and SuhB) represses expression of suhB and possibly other genes\n",
      "    via boxA; the Nus complex prevents or promotes Rho-mediated\n",
      "    transcription termination depending on gene context (PubMed:29229908).\n",
      "    Involved in post-transcriptional control of gene expression (Probable).\n",
      "    Enzymatic activity is not required for complementation of the cold-\n",
      "    sensitive phenotype of the dnaB121 mutation (PubMed:10747806)\n",
      "    (Probable). {ECO:0000269|PubMed:10747806, ECO:0000269|PubMed:26980831,\n",
      "    ECO:0000269|PubMed:29229908, ECO:0000269|PubMed:31020314,\n",
      "    ECO:0000269|PubMed:32871103, ECO:0000305|PubMed:17652087,\n",
      "    ECO:0000305|PubMed:1847383, ECO:0000305|PubMed:2138605,\n",
      "    ECO:0000305|PubMed:6389495, ECO:0000305|PubMed:8589060,\n",
      "    ECO:0000305|PubMed:8831954}.\n",
      "-!- FUNCTION: Has D,L-inositol-1-monophosphatase and beta-\n",
      "    glycerophosphatase activity, has less to no activity against a number\n",
      "    of other substrates (PubMed:8002619). 2.5-fold more active on 1D-\n",
      "    inositol-1-monophosphate than L-inositol-1-monophosphate (1D-myo-\n",
      "    inositol 3-phosphate). Specific activity increases significantly upon\n",
      "    heating. Only beta-glycerophosphate and adenosine 2'-monophosphate are\n",
      "    alternative substrates (PubMed:10747806). {ECO:0000269|PubMed:10747806,\n",
      "    ECO:0000269|PubMed:8002619}.\n",
      "-!- FUNCTION: Required for growth at low temperatures (PubMed:2138605,\n",
      "    PubMed:6389495, PubMed:1847383, PubMed:8002619, PubMed:8589060,\n",
      "    PubMed:17652087, PubMed:26980831). Identified as a suppressor (ssyA3)\n",
      "    of a temperature-sensitive, protein export missense mutation of secY\n",
      "    (secY24), allows growth at 42 but not 30 degrees Celsius\n",
      "    (PubMed:6389495). Identified as a suppressor (suhB2) of an rpoH\n",
      "    missense mutation (rpoH15), allowing growth at 37 and 40 but not 25, 30\n",
      "    or 34 degrees Celsius, increases expression of RpoH (PubMed:2138605).\n",
      "    Identified as a suppressor of a dnaB helicase missense mutation\n",
      "    (dnaB121), restores growth at 42 but not 30 degrees Celsius\n",
      "    (PubMed:1847383). In both suhB2 and ssyA3 there is an insertion in the\n",
      "    5' region of the gene which prevents SuhB protein expression\n",
      "    (PubMed:8002619, PubMed:8589060). Missense mutant suhB10 is suppressed\n",
      "    by mutations in RNase III (rnc), showing genetic interaction between\n",
      "    them (PubMed:8589060). Deletion of suhB is suppressed by mutations in\n",
      "    RNase III, by a mutation in nusA or deletion of nusB, indicating that\n",
      "    in the absence of SuhB the Nus complex inhibits growth\n",
      "    (PubMed:26980831). {ECO:0000269|PubMed:17652087,\n",
      "    ECO:0000269|PubMed:1847383, ECO:0000269|PubMed:2138605,\n",
      "    ECO:0000269|PubMed:26980831, ECO:0000269|PubMed:6389495,\n",
      "    ECO:0000269|PubMed:8002619, ECO:0000269|PubMed:8589060}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a myo-inositol phosphate + H2O = myo-inositol + phosphate;\n",
      "      Xref=Rhea:RHEA:24056, ChEBI:CHEBI:15377, ChEBI:CHEBI:17268,\n",
      "      ChEBI:CHEBI:43474, ChEBI:CHEBI:84139; EC=3.1.3.25;\n",
      "      Evidence={ECO:0000269|PubMed:10747806, ECO:0000269|PubMed:8002619};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=1D-myo-inositol 1-phosphate + H2O = myo-inositol + phosphate;\n",
      "      Xref=Rhea:RHEA:27670, ChEBI:CHEBI:15377, ChEBI:CHEBI:17268,\n",
      "      ChEBI:CHEBI:43474, ChEBI:CHEBI:58433; EC=3.1.3.25;\n",
      "      Evidence={ECO:0000269|PubMed:10747806, ECO:0000269|PubMed:17652087};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=1D-myo-inositol 3-phosphate + H2O = myo-inositol + phosphate;\n",
      "      Xref=Rhea:RHEA:30739, ChEBI:CHEBI:15377, ChEBI:CHEBI:17268,\n",
      "      ChEBI:CHEBI:43474, ChEBI:CHEBI:58401; EC=3.1.3.25;\n",
      "      Evidence={ECO:0000269|PubMed:10747806};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:10747806, ECO:0000269|PubMed:31020314,\n",
      "      ECO:0000269|PubMed:32871103, ECO:0000269|PubMed:8002619};\n",
      "    Note=Partial activity is seen with Co(2+), Ni(2+), Mn(2+), Zn(2+) and\n",
      "    Fe(2+); in the presence of Mg(2+) these cations inhibit.\n",
      "    {ECO:0000269|PubMed:10747806};\n",
      "-!- ACTIVITY REGULATION: Inhibited by Li(+) (PubMed:8002619,\n",
      "    PubMed:10747806) (Probable). Li(+) binds to Asp-84, Asp-87 and Asp-212\n",
      "    (PubMed:22384802). {ECO:0000269|PubMed:10747806,\n",
      "    ECO:0000269|PubMed:22384802, ECO:0000269|PubMed:8002619,\n",
      "    ECO:0000305|PubMed:22384802}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=71 uM for myo-inositol phosphate {ECO:0000269|PubMed:8002619};\n",
      "      KM=64 uM for 1D-myo-inositol 1-phosphate\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "      KM=79 uM for 1D-myo-inositol 3-phosphate\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "      KM=69 uM for myo-inositol phosphate {ECO:0000269|PubMed:10747806};\n",
      "      KM=92 uM for myo-inositol phosphate after 5 minutes at 70 degrees\n",
      "      Celsius {ECO:0000269|PubMed:10747806};\n",
      "      KM=110 uM for 1D-myo-inositol 1-phosphate\n",
      "      {ECO:0000269|PubMed:17652087};\n",
      "      Vmax=12.3 umol/min/mg enzyme {ECO:0000269|PubMed:8002619};\n",
      "      Vmax=6.9 umol/min/mg enzyme for 1D-myo-inositol 1-phosphate\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "      Vmax=2.66 umol/min/mg enzyme for 1D-myo-inositol 3-phosphate\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "      Vmax=3.38 umol/min/mg enzyme for myo-inositol phosphate\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "      Vmax=27.6 umol/min/mg enzyme for myo-inositol phosphate after 5\n",
      "      minutes at 70 degrees Celsius {ECO:0000269|PubMed:10747806};\n",
      "    pH dependence:\n",
      "      Optimum pH is 7.8. {ECO:0000269|PubMed:8002619};\n",
      "    Temperature dependence:\n",
      "      Optimum temperature is 80 degrees Celsius, 90% of activity remains\n",
      "      after heating at 70 degrees Celsius for 5 minutes.\n",
      "      {ECO:0000269|PubMed:10747806};\n",
      "-!- SUBUNIT: Monomer (PubMed:10747806, PubMed:31127279). A monomer-dimer\n",
      "    equilibrium (PubMed:17652087). Homodimer (PubMed:31020314,\n",
      "    PubMed:32871103). The rRNA transcription and antitermination complex\n",
      "    (rrnTAC) consists of RNA polymerase (RNAP), NusA, NusB, NusE (rpsJ),\n",
      "    NusG, SubB, ribosomal protein S4, DNA and precursor rRNA; S4 is more\n",
      "    flexible than other subunits (PubMed:31127279, PubMed:31020314,\n",
      "    PubMed:32871103). Binds to RNAP, which decreases IMPase activity\n",
      "    against 1D-myo-inositol 1-phosphate (PubMed:17652087) (Probable).\n",
      "    Association with rRNA gene-transcribing RNAP is dependent on NusB\n",
      "    (PubMed:26980831, PubMed:29229908). Interacts with AR2 domain of NusA;\n",
      "    crystallizes as a 2:1 SuhB:NusA heterotrimer (PubMed:31020314,\n",
      "    PubMed:31127279). {ECO:0000269|PubMed:10747806,\n",
      "    ECO:0000269|PubMed:17652087, ECO:0000269|PubMed:26980831,\n",
      "    ECO:0000269|PubMed:29229908, ECO:0000269|PubMed:31020314,\n",
      "    ECO:0000269|PubMed:31127279, ECO:0000269|PubMed:32871103,\n",
      "    ECO:0000305|PubMed:31127279}.\n",
      "Localização subcelular:  Cytoplasm {ECO:0000305|PubMed:10747806}.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305|PubMed:10747806}.\n",
      "-!- INDUCTION: Transcription is autoregulated (PubMed:8831954). Repressed\n",
      "    by the Nus factor complex (NusA, NusB, NusE (rpsJ), NusG and SuhB)\n",
      "    (PubMed:29229908). {ECO:0000269|PubMed:29229908,\n",
      "    ECO:0000269|PubMed:8831954}.\n",
      "-!- DISRUPTION PHENOTYPE: Conditionally lethal, cells are unable to grow at\n",
      "    30 degrees Celsius and less, grow poorly at 37 degrees Celsius, but do\n",
      "    grow at 42 degrees Celsius. {ECO:0000269|PubMed:17652087,\n",
      "    ECO:0000269|PubMed:26980831}.\n",
      "-!- MISCELLANEOUS: E.coli makes very low amounts of myo-inositol-containing\n",
      "    phospholipids, so the catalytic necessity for this enzyme is low.\n",
      "    {ECO:0000305|PubMed:17652087, ECO:0000305|PubMed:1848220}.\n",
      "-!- SIMILARITY: Belongs to the inositol monophosphatase superfamily.\n",
      "    {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00629; IMP_1; 1.\n",
      "Domínios PROSITE: PROSITE; PS00630; IMP_2; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: LAMB_ECOLI  Reviewed; 446 AA.\n",
      "ID: P02943; Q2M6R8;\n",
      "RecName: Full=Maltoporin {ECO:0000255|HAMAP-Rule:MF_01301};\n",
      "AltName: Full=Maltose outer membrane channel {ECO:0000305};\n",
      "AltName: Full=Maltose-inducible porin {ECO:0000255|HAMAP-Rule:MF_01301};\n",
      "AltName: Full=Phage lambda receptor protein {ECO:0000303|PubMed:4201774};\n",
      "Flags: Precursor;\n",
      "Name=lamB {ECO:0000255|HAMAP-Rule:MF_01301}; Synonyms=malB;\n",
      "OrderedLocusNames=b4036, JW3996;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 1-156, FUNCTION, AND MUTAGENESIS OF\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 26-446, FUNCTION, DISRUPTION\n",
      "X-RAY CRYSTALLOGRAPHY (3.2 ANGSTROMS), FUNCTION, AND MUTAGENESIS OF\n",
      "X-RAY CRYSTALLOGRAPHY (3.1 ANGSTROMS), FUNCTION, SUBUNIT, DISULFIDE BOND,\n",
      "SUBSTRATES, FUNCTION, AND SUBUNIT.\n",
      "AND GLUCOSE, FUNCTION, AND SUBUNIT.\n",
      "-!- FUNCTION: Involved in the transport of maltose and maltodextrins\n",
      "    (PubMed:3301537, PubMed:2832377, PubMed:11742115, PubMed:7824948,\n",
      "    PubMed:8805519, PubMed:9299337). Indispensable for translocation of\n",
      "    maltodextrins (alpha 1-4 linked polyglucosyls) containing more than\n",
      "    three glucosyl moieties. A hydrophobic path ('greasy slide') of\n",
      "    aromatic residues serves to guide and select the sugars for transport\n",
      "    through the channel (PubMed:7824948, PubMed:8805519, PubMed:9299337,\n",
      "    PubMed:11742115). Also acts as a receptor for several bacteriophages\n",
      "    including lambda (PubMed:4201774). Binds maltosaccharides; when LamB\n",
      "    binds starch in soft agar, it inhibits motility (PubMed:2832377).\n",
      "    {ECO:0000269|PubMed:11742115, ECO:0000269|PubMed:2832377,\n",
      "    ECO:0000269|PubMed:3301537, ECO:0000269|PubMed:4201774,\n",
      "    ECO:0000269|PubMed:7824948, ECO:0000269|PubMed:8805519,\n",
      "    ECO:0000269|PubMed:9299337}.\n",
      "-!- SUBUNIT: Homotrimer formed of three 18-stranded antiparallel beta-\n",
      "    barrels, containing three independent channels (PubMed:1988451,\n",
      "    PubMed:16079137, PubMed:7824948, PubMed:8805519, PubMed:9299337) (By\n",
      "    similarity). An intrasubunit disulfide bond contributes to trimer\n",
      "    stability (PubMed:1988451). {ECO:0000255|HAMAP-Rule:MF_01301,\n",
      "    ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:1988451,\n",
      "    ECO:0000269|PubMed:7824948, ECO:0000269|PubMed:8805519,\n",
      "    ECO:0000269|PubMed:9299337}.\n",
      "-!- INTERACTION:\n",
      "    P02943; P10408: secA; NbExp=3; IntAct=EBI-371309, EBI-543213;\n",
      "Localização subcelular:  Cell outer membrane {ECO:0000255|HAMAP-\n",
      "-!- SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-\n",
      "    Rule:MF_01301, ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:1988451,\n",
      "    ECO:0000269|PubMed:4201774, ECO:0000269|Ref.9}; Multi-pass membrane\n",
      "    protein {ECO:0000255|HAMAP-Rule:MF_01301, ECO:0000269|PubMed:16079137}.\n",
      "-!- INDUCTION: By maltose. {ECO:0000255|HAMAP-Rule:MF_01301,\n",
      "    ECO:0000269|Ref.9}.\n",
      "-!- DISRUPTION PHENOTYPE: Resistance to phage lambda (PubMed:4201774). No\n",
      "    growth on dextrins (PubMed:2832377). {ECO:0000269|PubMed:2832377,\n",
      "    ECO:0000269|PubMed:4201774}.\n",
      "-!- SIMILARITY: Belongs to the porin LamB (TC 1.B.3) family.\n",
      "    {ECO:0000255|HAMAP-Rule:MF_01301}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: DLD_ECOLI   Reviewed; 571 AA.\n",
      "ID: P06149; Q2MAU6;\n",
      "RecName: Full=Quinone-dependent D-lactate dehydrogenase {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000305};\n",
      "         EC=1.1.5.12 {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000269|PubMed:3013300, ECO:0000269|PubMed:7578233};\n",
      "AltName: Full=(R)-lactate:quinone 2-oxidoreductase {ECO:0000305};\n",
      "AltName: Full=D-lactate dehydrogenase {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000303|PubMed:4575624, ECO:0000303|PubMed:4582730};\n",
      "         Short=D-LDH {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000303|PubMed:3882663};\n",
      "AltName: Full=Respiratory D-lactate dehydrogenase {ECO:0000305};\n",
      "Name=dld {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000303|PubMed:6386470};\n",
      "OrderedLocusNames=b2133, JW2121;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, COFACTOR, AND SUBCELLULAR LOCATION.\n",
      "FUNCTION, COFACTOR, ACTIVITY REGULATION, AND SUBCELLULAR LOCATION.\n",
      "FUNCTION, AND CATALYTIC ACTIVITY.\n",
      "FUNCTION, AND BIOPHYSICOCHEMICAL PROPERTIES.\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND COFACTOR.\n",
      "-!- FUNCTION: Catalyzes the oxidation of D-lactate to pyruvate. Electrons\n",
      "    derived from D-lactate oxidation are transferred to the\n",
      "    ubiquinone/cytochrome electron transfer chain, where they may be used\n",
      "    to provide energy for the active transport of a variety of amino acids\n",
      "    and sugars across the membrane. {ECO:0000269|PubMed:2185834,\n",
      "    ECO:0000269|PubMed:3013300, ECO:0000269|PubMed:4575624,\n",
      "    ECO:0000269|PubMed:4582730, ECO:0000269|PubMed:7578233}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=(R)-lactate + a quinone = a quinol + pyruvate;\n",
      "      Xref=Rhea:RHEA:51468, ChEBI:CHEBI:15361, ChEBI:CHEBI:16004,\n",
      "      ChEBI:CHEBI:24646, ChEBI:CHEBI:132124; EC=1.1.5.12;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_02092,\n",
      "      ECO:0000269|PubMed:3013300, ECO:0000269|PubMed:7578233};\n",
      "-!- COFACTOR:\n",
      "    Name=FAD; Xref=ChEBI:CHEBI:57692;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_02092,\n",
      "      ECO:0000269|PubMed:10944213, ECO:0000269|PubMed:4575624,\n",
      "      ECO:0000269|PubMed:4582730, ECO:0000269|PubMed:7578233};\n",
      "-!- ACTIVITY REGULATION: Inhibited by 2-hydroxy-3-butynoic acid, but not by\n",
      "    p-chloromercuribenzoate, n-ethylmaleimide, or 5,5'-dithiobis(2-\n",
      "    nitrobenzoic acid). {ECO:0000269|PubMed:4582730}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=0.16 mM for D-lactate {ECO:0000269|PubMed:2185834};\n",
      "Localização subcelular:  Cell inner membrane {ECO:0000255|HAMAP-\n",
      "-!- SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-\n",
      "    Rule:MF_02092, ECO:0000269|PubMed:1092688, ECO:0000269|PubMed:4575624,\n",
      "    ECO:0000269|PubMed:4582730}; Peripheral membrane protein\n",
      "    {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000269|PubMed:1092688};\n",
      "    Cytoplasmic side {ECO:0000255|HAMAP-Rule:MF_02092,\n",
      "    ECO:0000269|PubMed:1092688}. Note=May bind the membrane through\n",
      "    electrostatic rather than hydrophobic forces.\n",
      "    {ECO:0000305|PubMed:10944213}.\n",
      "-!- DOMAIN: Contains 3 domains: the flavin adenine dinucleotide (FAD)-\n",
      "    binding domain, the cap domain and the membrane-binding domain.\n",
      "    {ECO:0000269|PubMed:10944213}.\n",
      "-!- SIMILARITY: Belongs to the quinone-dependent D-lactate dehydrogenase\n",
      "    family. {ECO:0000255|HAMAP-Rule:MF_02092, ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAA60530.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "Domínios PROSITE: PROSITE; PS51387; FAD_PCMH; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MOBA_ECOLI  Reviewed; 194 AA.\n",
      "ID: P32173; Q2M8F5; Q9LBV0;\n",
      "RecName: Full=Molybdenum cofactor guanylyltransferase;\n",
      "         Short=MoCo guanylyltransferase;\n",
      "         EC=2.7.7.77;\n",
      "AltName: Full=GTP:molybdopterin guanylyltransferase;\n",
      "AltName: Full=Mo-MPT guanylyltransferase;\n",
      "AltName: Full=Molybdopterin guanylyltransferase;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide biosynthesis protein A;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide synthase;\n",
      "         Short=MGD synthase;\n",
      "AltName: Full=Protein FA;\n",
      "Name=mobA; Synonyms=chlB, mob, narB; OrderedLocusNames=b3857, JW3829;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 1-5, FUNCTION IN MGD BIOSYNTHESIS, AND SUBUNIT.\n",
      "FUNCTION IN MGD BIOSYNTHESIS, AND DISRUPTION PHENOTYPE.\n",
      "FUNCTION IN BIS(MGD) AND MGD BIOSYNTHESIS, CATALYTIC ACTIVITY, SUBSTRATE\n",
      "FUNCTION, CATALYTIC ACTIVITY, KINETIC PARAMETERS, DOMAIN, AND MUTAGENESIS\n",
      "-!- FUNCTION: Transfers a GMP moiety from GTP to Mo-molybdopterin (Mo-MPT)\n",
      "    cofactor (Moco or molybdenum cofactor) to form Mo-molybdopterin guanine\n",
      "    dinucleotide (Mo-MGD) cofactor. Is also involved in the biosynthesis of\n",
      "    the bis-MGD form of the Moco cofactor (Mo-bisMGD) in which the metal is\n",
      "    symmetrically ligated by the dithiolene groups of two MGD molecules. Is\n",
      "    necessary and sufficient for the in vitro activation of the DMSOR\n",
      "    molybdoenzyme that uses the Mo-bisMGD form of molybdenum cofactor,\n",
      "    which implies formation and efficient insertion of the cofactor into\n",
      "    the enzyme without the need of a chaperone. Is specific for GTP since\n",
      "    other nucleotides such as ATP and GMP cannot be utilized.\n",
      "    {ECO:0000269|PubMed:10978348, ECO:0000269|PubMed:1648082,\n",
      "    ECO:0000269|PubMed:21081498, ECO:0000269|PubMed:8020507}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=GTP + H(+) + Mo-molybdopterin = diphosphate + Mo-molybdopterin\n",
      "      guanine dinucleotide; Xref=Rhea:RHEA:34243, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:37565, ChEBI:CHEBI:71302,\n",
      "      ChEBI:CHEBI:71310; EC=2.7.7.77;\n",
      "      Evidence={ECO:0000269|PubMed:10978348, ECO:0000269|PubMed:21081498};\n",
      "-!- COFACTOR:\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:10978347, ECO:0000269|PubMed:10978348};\n",
      "    Name=Mn(2+); Xref=ChEBI:CHEBI:29035;\n",
      "      Evidence={ECO:0000269|PubMed:10978347, ECO:0000269|PubMed:10978348};\n",
      "    Note=Both divalent cations appear to be equally efficient in an vitro\n",
      "    reconstitution assay. {ECO:0000269|PubMed:10978347,\n",
      "    ECO:0000269|PubMed:10978348};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=6.5 uM for GTP {ECO:0000269|PubMed:21081498};\n",
      "-!- SUBUNIT: Monomer. An equilibrium exists between a monomeric and\n",
      "    oligomeric form of the enzyme, which could be an octamer; whether this\n",
      "    oligomeric arrangement is of functional relevance is unclear. Interacts\n",
      "    with MoeA and MobB in vivo. {ECO:0000269|PubMed:10978347,\n",
      "    ECO:0000269|PubMed:12372836, ECO:0000269|PubMed:8020507}.\n",
      "-!- INTERACTION:\n",
      "    P32173; P12281: moeA; NbExp=3; IntAct=EBI-1133881, EBI-554393;\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- INDUCTION: Is expressed at very low levels under both aerobic and\n",
      "    anaerobic growth conditions. {ECO:0000269|PubMed:7551035}.\n",
      "-!- DOMAIN: The N-terminal domain determines nucleotide recognition and\n",
      "    specific binding, while the C-terminal domain determines the specific\n",
      "    binding to the target protein. When the N-terminal domain of MobA is\n",
      "    fused to the C-terminal domain of MocA, comparable kinetic constants as\n",
      "    wild-type MobA are obtained with GTP, and the activity with CTP is\n",
      "    completely lost. Consistent results are obtained when the N-terminal\n",
      "    domain of MocA is fused to the C-terminal domain of MobA: the kinetic\n",
      "    constants with CTP are comparable with the ones found for wild-type\n",
      "    MocA, although no activity with GTP is detected.\n",
      "    {ECO:0000269|PubMed:21081498}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene are chlorate-resistant,\n",
      "    fail to synthesize MGD and accumulate elevated quantities of MPT.\n",
      "    {ECO:0000269|PubMed:1648082}.\n",
      "-!- SIMILARITY: Belongs to the MobA family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: MOBB_ECOLI  Reviewed; 175 AA.\n",
      "ID: P32125; P76770; Q2M8F4;\n",
      "RecName: Full=Molybdopterin-guanine dinucleotide biosynthesis adapter protein;\n",
      "         Short=MGD biosynthesis adapter protein;\n",
      "AltName: Full=Molybdenum cofactor biosynthesis adapter protein;\n",
      "         Short=Moco biosynthesis adapter protein;\n",
      "AltName: Full=Molybdopterin-guanine dinucleotide biosynthesis protein B;\n",
      "Name=mobB; Synonyms=yihC; OrderedLocusNames=b3856, JW5575;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 2-13, GTP-BINDING, GTPASE ACTIVITY, FUNCTION, AND\n",
      "X-RAY CRYSTALLOGRAPHY (1.90 ANGSTROMS) IN COMPLEX WITH SULFATE, FUNCTION,\n",
      "-!- FUNCTION: GTP-binding protein that is not required for the biosynthesis\n",
      "    of Mo-molybdopterin guanine dinucleotide (Mo-MGD) cofactor, and not\n",
      "    necessary for the formation of active molybdoenzymes using this form of\n",
      "    molybdenum cofactor. May act as an adapter protein to achieve the\n",
      "    efficient biosynthesis and utilization of MGD. Displays a weak\n",
      "    intrinsic GTPase activity. Is also able to bind the nucleotides ATP,\n",
      "    TTP and GDP, but with lower affinity than GTP.\n",
      "    {ECO:0000269|PubMed:12682065, ECO:0000269|PubMed:9219527}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=7.5 uM for GTP;\n",
      "      Note=kcat is 0.003 min(-1) for the GTPase activity.;\n",
      "-!- SUBUNIT: Homodimer. Interacts with MobA, MogA and MoeA in vivo.\n",
      "    {ECO:0000269|PubMed:12372836, ECO:0000269|PubMed:12682065,\n",
      "    ECO:0000269|PubMed:14646116, ECO:0000269|PubMed:9219527}.\n",
      "-!- INTERACTION:\n",
      "    P32125; P30749: moaE; NbExp=2; IntAct=EBI-879965, EBI-554376;\n",
      "-!- INDUCTION: Is expressed at very low levels under both aerobic and\n",
      "    anaerobic growth conditions. {ECO:0000269|PubMed:7551035}.\n",
      "-!- SIMILARITY: Belongs to the MobB family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAB02991.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "    Sequence=BAE77452.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: GLO2_ECOLI  Reviewed; 251 AA.\n",
      "ID: P0AC84; Q47677;\n",
      "RecName: Full=Hydroxyacylglutathione hydrolase GloB {ECO:0000305|PubMed:17196158, ECO:0000305|PubMed:25670698};\n",
      "         EC=3.1.2.6 {ECO:0000269|PubMed:17196158, ECO:0000269|PubMed:25670698};\n",
      "AltName: Full=Glyoxalase II {ECO:0000303|PubMed:17196158, ECO:0000303|PubMed:25670698};\n",
      "         Short=Glx II {ECO:0000303|PubMed:17196158, ECO:0000303|PubMed:25670698};\n",
      "Name=gloB {ECO:0000303|PubMed:25670698}; Synonyms=yafR;\n",
      "OrderedLocusNames=b0212, JW0202;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, CATALYTIC ACTIVITY, BIOPHYSICOCHEMICAL PROPERTIES, COFACTOR,\n",
      "FUNCTION, CATALYTIC ACTIVITY, COFACTOR, BIOPHYSICOCHEMICAL PROPERTIES,\n",
      "-!- FUNCTION: Type II glyoxalase that catalyzes the hydrolysis of (R)-S-\n",
      "    lactoylglutathione to (R)-lactate and glutathione (PubMed:25670698,\n",
      "    PubMed:17196158). Is more efficient than the isozyme GloC, and plays a\n",
      "    major contribution to methylglyoxal (MG) detoxification in E.coli\n",
      "    (PubMed:25670698). The two isoenzymes have additive effects and ensure\n",
      "    maximal MG degradation (PubMed:25670698). {ECO:0000269|PubMed:17196158,\n",
      "    ECO:0000269|PubMed:25670698}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=an S-(2-hydroxyacyl)glutathione + H2O = a 2-hydroxy\n",
      "      carboxylate + glutathione + H(+); Xref=Rhea:RHEA:21864,\n",
      "      ChEBI:CHEBI:15377, ChEBI:CHEBI:15378, ChEBI:CHEBI:57925,\n",
      "      ChEBI:CHEBI:58896, ChEBI:CHEBI:71261; EC=3.1.2.6;\n",
      "      Evidence={ECO:0000269|PubMed:17196158, ECO:0000269|PubMed:25670698};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=(R)-S-lactoylglutathione + H2O = (R)-lactate + glutathione +\n",
      "      H(+); Xref=Rhea:RHEA:25245, ChEBI:CHEBI:15377, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:16004, ChEBI:CHEBI:57474, ChEBI:CHEBI:57925; EC=3.1.2.6;\n",
      "      Evidence={ECO:0000269|PubMed:17196158, ECO:0000269|PubMed:25670698};\n",
      "-!- COFACTOR:\n",
      "    Name=Zn(2+); Xref=ChEBI:CHEBI:29105;\n",
      "      Evidence={ECO:0000269|PubMed:17196158, ECO:0000305|PubMed:25670698};\n",
      "    Note=Binds 2 Zn(2+) ions per subunit. Mn(2+) and Co(2+) can substitute\n",
      "    for zinc in reconstitution experiments. {ECO:0000269|PubMed:17196158};\n",
      "-!- ACTIVITY REGULATION: Is inhibited by Cu(2+).\n",
      "    {ECO:0000269|PubMed:25670698}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=184 uM for (R)-S-lactoylglutathione {ECO:0000269|PubMed:17196158};\n",
      "      KM=0.5 mM for (R)-S-lactoylglutathione {ECO:0000269|PubMed:25670698};\n",
      "      Vmax=112 umol/min/mg enzyme for the hydrolysis of (R)-S-\n",
      "      lactoylglutathione {ECO:0000269|PubMed:17196158};\n",
      "      Note=kcat is 53 sec(-1) for the hydrolysis of (R)-S-\n",
      "      lactoylglutathione (PubMed:17196158). kcat is 15.6 sec(-1) for the\n",
      "      hydrolysis of (R)-S-lactoylglutathione (PubMed:25670698).\n",
      "      {ECO:0000269|PubMed:17196158, ECO:0000269|PubMed:25670698};\n",
      "-!- PATHWAY: Secondary metabolite metabolism; methylglyoxal degradation;\n",
      "    (R)-lactate from methylglyoxal: step 2/2.\n",
      "    {ECO:0000269|PubMed:25670698}.\n",
      "-!- SUBUNIT: Monomer. {ECO:0000269|PubMed:17196158}.\n",
      "-!- MASS SPECTROMETRY: Mass=28432; Method=Electrospray;\n",
      "    Evidence={ECO:0000269|PubMed:17196158};\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene show decreased\n",
      "    methylglyoxal tolerance. A double deletion mutant lacking both gloB and\n",
      "    gloC exhibits almost no resistance to exogenously supplied\n",
      "    methylglyoxal, and is unable to grow at MG concentrations as low as 0.1\n",
      "    mM. {ECO:0000269|PubMed:25670698}.\n",
      "-!- SIMILARITY: Belongs to the metallo-beta-lactamase superfamily.\n",
      "    Glyoxalase II family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: OMPN_ECOLI  Reviewed; 377 AA.\n",
      "ID: P77747; P76854;\n",
      "RecName: Full=Outer membrane porin N;\n",
      "AltName: Full=Outer membrane protein N;\n",
      "AltName: Full=Porin OmpN;\n",
      "Flags: Precursor;\n",
      "Name=ompN; Synonyms=ynaG; OrderedLocusNames=b1377, JW1371;\n",
      "Source: Escherichia coli (strain K12).\n",
      "-!- FUNCTION: Forms pores that allow passive diffusion of small molecules\n",
      "    across the outer membrane (By similarity). Non-specific porin.\n",
      "    {ECO:0000250}.\n",
      "-!- SUBUNIT: Homotrimer.\n",
      "Localização subcelular:  Cell outer membrane; Multi-pass membrane protein.\n",
      "-!- SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membrane protein.\n",
      "-!- SIMILARITY: Belongs to the Gram-negative porin family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: PHOE_ECOLI  Reviewed; 351 AA.\n",
      "ID: P02932;\n",
      "RecName: Full=Outer membrane porin PhoE;\n",
      "AltName: Full=Outer membrane pore protein E;\n",
      "Flags: Precursor;\n",
      "Name=phoE; Synonyms=ompE; OrderedLocusNames=b0241, JW0231;\n",
      "Source: Escherichia coli (strain K12).\n",
      "-!- FUNCTION: Uptake of inorganic phosphate, phosphorylated compounds, and\n",
      "    some other negatively charged solutes.\n",
      "-!- SUBUNIT: Homotrimer (PubMed:2464593). Forms mixed heterotrimers with\n",
      "    OmpC and with OmpF; other mixed heterotrimers are also probable\n",
      "    (PubMed:2464593). {ECO:0000269|PubMed:2464593}.\n",
      "Localização subcelular:  Cell outer membrane {ECO:0000269|PubMed:2464593};\n",
      "-!- SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000269|PubMed:2464593};\n",
      "    Multi-pass membrane protein.\n",
      "-!- INDUCTION: By phosphate starvation. {ECO:0000269|PubMed:2464593}.\n",
      "-!- SIMILARITY: Belongs to the Gram-negative porin family. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00576; GRAM_NEG_PORIN; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: OMPF_ECOLI  Reviewed; 362 AA.\n",
      "ID: P02931;\n",
      "RecName: Full=Outer membrane porin F;\n",
      "AltName: Full=Outer membrane protein 1A;\n",
      "AltName: Full=Outer membrane protein B;\n",
      "AltName: Full=Outer membrane protein F;\n",
      "AltName: Full=Outer membrane protein IA;\n",
      "AltName: Full=Porin OmpF;\n",
      "Flags: Precursor;\n",
      "Name=ompF; Synonyms=cmlB, coa, cry, tolF; OrderedLocusNames=b0929, JW0912;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION (MICROBIAL INFECTION), AND DISRUPTION PHENOTYPE.\n",
      "-!- FUNCTION: Forms pores that allow passive diffusion of small molecules\n",
      "    across the outer membrane. {ECO:0000305|PubMed:19721064}.\n",
      "-!- FUNCTION: (Microbial infection) It is also a receptor for the\n",
      "    bacteriophage T2. Is the major receptor for colicin E5 (Probable).\n",
      "    {ECO:0000305|PubMed:27723824}.\n",
      "-!- FUNCTION: (Microbial infection) A mixed OmpC-OmpF heterotrimer is the\n",
      "    outer membrane receptor for toxin CdiA-EC536; polymorphisms in\n",
      "    extracellular loops 4 and 5 of OmpC confer susceptibility to CdiA-\n",
      "    EC536-mediated toxicity. {ECO:0000269|PubMed:27723824}.\n",
      "-!- SUBUNIT: Homotrimer (PubMed:2464593, PubMed:16079137). Forms mixed\n",
      "    heterotrimers with OmpC and with PhoE; other mixed heterotrimers are\n",
      "    also probable (PubMed:2464593). {ECO:0000269|PubMed:16079137,\n",
      "    ECO:0000269|PubMed:2464593}.\n",
      "-!- INTERACTION:\n",
      "    P02931; P76506: mlaA; NbExp=4; IntAct=EBI-371336, EBI-1128511;\n",
      "    P02931; P02931: ompF; NbExp=7; IntAct=EBI-371336, EBI-371336;\n",
      "    P02931; P08083: cna; Xeno; NbExp=3; IntAct=EBI-371336, EBI-15691934;\n",
      "    P02931; P09883: col; Xeno; NbExp=4; IntAct=EBI-371336, EBI-1029888;\n",
      "Localização subcelular:  Cell outer membrane {ECO:0000269|PubMed:16079137,\n",
      "-!- SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000269|PubMed:16079137,\n",
      "    ECO:0000269|PubMed:2464593}; Multi-pass membrane protein\n",
      "    {ECO:0000269|PubMed:16079137}.\n",
      "-!- INDUCTION: By growth in low osmolarity conditions (low sugar or salt);\n",
      "    activated at low osmolarity by OmpR, repressed at high osmolarity by\n",
      "    OmpR (at protein level) (PubMed:3010044, PubMed:2464593). Levels of\n",
      "    OmpF protein decrease in a lon/ycgE double disruption (at protein\n",
      "    level) (PubMed:19721064). {ECO:0000269|PubMed:19721064,\n",
      "    ECO:0000269|PubMed:2464593, ECO:0000269|PubMed:3010044}.\n",
      "-!- DISRUPTION PHENOTYPE: Leads to decreased susceptibility to a number of\n",
      "    hydrophilic antibiotics including ampicillin, cefoxitin and\n",
      "    tetracycline (PubMed:19721064). Deletion of ompF confers resistance to\n",
      "    colicin E5 (PubMed:27723824). {ECO:0000269|PubMed:19721064,\n",
      "    ECO:0000269|PubMed:27723824}.\n",
      "-!- SIMILARITY: Belongs to the Gram-negative porin family. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00576; GRAM_NEG_PORIN; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: OMPC_ECOLI  Reviewed; 367 AA.\n",
      "ID: P06996;\n",
      "RecName: Full=Outer membrane porin C;\n",
      "AltName: Full=Outer membrane protein 1B;\n",
      "AltName: Full=Outer membrane protein C;\n",
      "AltName: Full=Porin OmpC;\n",
      "Flags: Precursor;\n",
      "Name=ompC; Synonyms=meoA, par; OrderedLocusNames=b2215, JW2203;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION (MICROBIAL INFECTION), AND DISRUPTION PHENOTYPE.\n",
      "-!- FUNCTION: Forms pores that allow passive diffusion of small molecules\n",
      "    across the outer membrane.\n",
      "-!- FUNCTION: (Microbial infection) Supports colicin E5 entry in the\n",
      "    absence of its major receptor OmpF. {ECO:0000305|PubMed:27723824}.\n",
      "-!- FUNCTION: (Microbial infection) A mixed OmpC-OmpF heterotrimer is the\n",
      "    outer membrane receptor for toxin CdiA-EC536; polymorphisms in\n",
      "    extracellular loops 4 and 5 of OmpC confer susceptibility to CdiA-\n",
      "    EC536-mediated toxicity. {ECO:0000269|PubMed:27723824}.\n",
      "-!- SUBUNIT: Homotrimer (PubMed:2464593, PubMed:16949612). Forms mixed\n",
      "    heterotrimers with OmpF and with PhoE; other mixed heterotrimers are\n",
      "    also probable (PubMed:2464593). {ECO:0000269|PubMed:16949612,\n",
      "    ECO:0000269|PubMed:2464593}.\n",
      "-!- INTERACTION:\n",
      "    P06996; P0C0V0: degP; NbExp=7; IntAct=EBI-371155, EBI-547165;\n",
      "    P06996; P76506: mlaA; NbExp=4; IntAct=EBI-371155, EBI-1128511;\n",
      "Localização subcelular:  Cell outer membrane {ECO:0000269|PubMed:2464593};\n",
      "-!- SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000269|PubMed:2464593};\n",
      "    Multi-pass membrane protein {ECO:0000269|PubMed:16949612}.\n",
      "-!- INDUCTION: By growth in high osmolarity conditions (high sugar or salt)\n",
      "    under control of OmpR (at protein level) (PubMed:3010044,\n",
      "    PubMed:2464593). In the presence of 0.2 M NaCl, 2.0 mM sodium cholate\n",
      "    (bile salts) decreases expression from the ompC promoter; how this is\n",
      "    mediated is unknown (PubMed:28423182). {ECO:0000269|PubMed:2464593,\n",
      "    ECO:0000269|PubMed:28423182, ECO:0000269|PubMed:3010044}.\n",
      "-!- DOMAIN: Loop L3 (residues 116-133) extends along the inner side of the\n",
      "    beta barrel wall and may constrict the pore mid-length.\n",
      "    {ECO:0000269|PubMed:16949612}.\n",
      "-!- DISRUPTION PHENOTYPE: Deletion of both ompC and ompF confers resistance\n",
      "    to colicin E5 and to the toxic activity of CdiA-EC536.\n",
      "    {ECO:0000269|PubMed:27723824}.\n",
      "-!- MISCELLANEOUS: Binds 1 Mg(2+) per subunit; could be Ca(2+) in vivo.\n",
      "-!- SIMILARITY: Belongs to the Gram-negative porin family. {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS00576; GRAM_NEG_PORIN; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: NADA_ECOLI  Reviewed; 347 AA.\n",
      "ID: P11458; P77373;\n",
      "RecName: Full=Quinolinate synthase {ECO:0000255|HAMAP-Rule:MF_00567, ECO:0000303|PubMed:18651751};\n",
      "         EC=2.5.1.72 {ECO:0000255|HAMAP-Rule:MF_00567, ECO:0000269|PubMed:10648170, ECO:0000269|PubMed:15898769, ECO:0000269|PubMed:15967443, ECO:0000269|PubMed:18674537};\n",
      "AltName: Full=Quinolinate synthetase A {ECO:0000303|PubMed:2841129};\n",
      "Name=nadA {ECO:0000255|HAMAP-Rule:MF_00567, ECO:0000303|PubMed:2841129};\n",
      "Synonyms=nicA; OrderedLocusNames=b0750, JW0733;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 1-45, MASS SPECTROMETRY, FUNCTION, CATALYTIC ACTIVITY,\n",
      "FUNCTION, CATALYTIC ACTIVITY, COFACTOR, AND SUBUNIT.\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND COFACTOR.\n",
      "FUNCTION, CATALYTIC ACTIVITY, COFACTOR, SUBUNIT, DISULFIDE BOND, AND\n",
      "-!- FUNCTION: Catalyzes the condensation of iminoaspartate with\n",
      "    dihydroxyacetone phosphate to form quinolinate. {ECO:0000255|HAMAP-\n",
      "    Rule:MF_00567, ECO:0000269|PubMed:10648170,\n",
      "    ECO:0000269|PubMed:15898769, ECO:0000269|PubMed:15967443,\n",
      "    ECO:0000269|PubMed:18674537}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=dihydroxyacetone phosphate + iminosuccinate = H(+) + 2 H2O +\n",
      "      phosphate + quinolinate; Xref=Rhea:RHEA:25888, ChEBI:CHEBI:15377,\n",
      "      ChEBI:CHEBI:15378, ChEBI:CHEBI:29959, ChEBI:CHEBI:43474,\n",
      "      ChEBI:CHEBI:57642, ChEBI:CHEBI:77875; EC=2.5.1.72;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "      ECO:0000269|PubMed:10648170, ECO:0000269|PubMed:15898769,\n",
      "      ECO:0000269|PubMed:15967443, ECO:0000269|PubMed:18674537};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:25889;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "      ECO:0000269|PubMed:10648170, ECO:0000269|PubMed:15898769,\n",
      "      ECO:0000269|PubMed:15967443, ECO:0000269|PubMed:18674537};\n",
      "-!- COFACTOR:\n",
      "    Name=[4Fe-4S] cluster; Xref=ChEBI:CHEBI:49883;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "      ECO:0000269|PubMed:15898769, ECO:0000269|PubMed:15967443,\n",
      "      ECO:0000269|PubMed:18674537};\n",
      "    Note=Binds 1 [4Fe-4S] cluster per subunit. {ECO:0000255|HAMAP-\n",
      "    Rule:MF_00567, ECO:0000269|PubMed:15898769,\n",
      "    ECO:0000269|PubMed:15967443, ECO:0000269|PubMed:18674537};\n",
      "-!- ACTIVITY REGULATION: Activity is greater under oxic conditions and is\n",
      "    regulated by the formation of a reversible disulfide-bond between Cys-\n",
      "    291 and Cys-294. {ECO:0000269|PubMed:18651751}.\n",
      "-!- PATHWAY: Cofactor biosynthesis; NAD(+) biosynthesis; quinolinate from\n",
      "    iminoaspartate: step 1/1. {ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "    ECO:0000269|PubMed:10648170}.\n",
      "-!- SUBUNIT: Monomer (PubMed:18674537). Homodimer (PubMed:15967443).\n",
      "    {ECO:0000269|PubMed:15967443, ECO:0000269|PubMed:18674537}.\n",
      "Localização subcelular:  Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00567,\n",
      "    ECO:0000305}.\n",
      "-!- MASS SPECTROMETRY: Mass=38246; Method=MALDI;\n",
      "    Evidence={ECO:0000269|PubMed:10648170};\n",
      "-!- SIMILARITY: Belongs to the quinolinate synthase family. Type 1\n",
      "    subfamily. {ECO:0000255|HAMAP-Rule:MF_00567, ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=CAA31216.1; Type=Frameshift; Evidence={ECO:0000305};\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: FRMA_ECOLI  Reviewed; 369 AA.\n",
      "ID: P25437; P75696; Q2MC68; Q47533;\n",
      "RecName: Full=S-(hydroxymethyl)glutathione dehydrogenase;\n",
      "         EC=1.1.1.284;\n",
      "AltName: Full=Alcohol dehydrogenase class-3;\n",
      "         EC=1.1.1.1;\n",
      "AltName: Full=Alcohol dehydrogenase class-III;\n",
      "AltName: Full=Glutathione-dependent formaldehyde dehydrogenase;\n",
      "         Short=FALDH;\n",
      "         Short=FDH;\n",
      "         Short=GSH-FDH;\n",
      "         EC=1.1.1.-;\n",
      "Name=frmA; Synonyms=adhC; OrderedLocusNames=b0356, JW0347;\n",
      "Source: Escherichia coli (strain K12).\n",
      "PROTEIN SEQUENCE OF 1-47, FUNCTION, COFACTOR, AND SUBUNIT.\n",
      "-!- FUNCTION: Has high formaldehyde dehydrogenase activity in the presence\n",
      "    of glutathione and catalyzes the oxidation of normal alcohols in a\n",
      "    reaction that is not GSH-dependent. In addition, hemithiolacetals other\n",
      "    than those formed from GSH, including omega-thiol fatty acids, also are\n",
      "    substrates. {ECO:0000269|PubMed:1731906}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=NADP(+) + S-(hydroxymethyl)glutathione = H(+) + NADPH + S-\n",
      "      formylglutathione; Xref=Rhea:RHEA:19981, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:57688, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349,\n",
      "      ChEBI:CHEBI:58758; EC=1.1.1.284;\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=NAD(+) + S-(hydroxymethyl)glutathione = H(+) + NADH + S-\n",
      "      formylglutathione; Xref=Rhea:RHEA:19985, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:57540, ChEBI:CHEBI:57688, ChEBI:CHEBI:57945,\n",
      "      ChEBI:CHEBI:58758; EC=1.1.1.284;\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a primary alcohol + NAD(+) = an aldehyde + H(+) + NADH;\n",
      "      Xref=Rhea:RHEA:10736, ChEBI:CHEBI:15378, ChEBI:CHEBI:15734,\n",
      "      ChEBI:CHEBI:17478, ChEBI:CHEBI:57540, ChEBI:CHEBI:57945; EC=1.1.1.1;\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=a secondary alcohol + NAD(+) = a ketone + H(+) + NADH;\n",
      "      Xref=Rhea:RHEA:10740, ChEBI:CHEBI:15378, ChEBI:CHEBI:17087,\n",
      "      ChEBI:CHEBI:35681, ChEBI:CHEBI:57540, ChEBI:CHEBI:57945; EC=1.1.1.1;\n",
      "-!- COFACTOR:\n",
      "    Name=Zn(2+); Xref=ChEBI:CHEBI:29105; Evidence={ECO:0000250};\n",
      "    Note=Binds 2 Zn(2+) ions per subunit. {ECO:0000250};\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000269|PubMed:1731906}.\n",
      "-!- INTERACTION:\n",
      "    P25437; P25437: frmA; NbExp=3; IntAct=EBI-556478, EBI-556478;\n",
      "Localização subcelular:  Cytoplasm.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm.\n",
      "-!- INDUCTION: Induced by formaldehyde and repressed by FrmR.\n",
      "    {ECO:0000269|PubMed:15466022}.\n",
      "-!- SIMILARITY: Belongs to the zinc-containing alcohol dehydrogenase\n",
      "    family. Class-III subfamily. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAB18081.1; Type=Frameshift; Evidence={ECO:0000305};\n",
      "    Sequence=BAA12834.1; Type=Frameshift; Evidence={ECO:0000305};\n",
      "Domínios PROSITE: PROSITE; PS00059; ADH_ZINC; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: PDXK_ECOLI  Reviewed; 283 AA.\n",
      "ID: P40191; P76964;\n",
      "RecName: Full=Pyridoxine/pyridoxal/pyridoxamine kinase {ECO:0000303|PubMed:8764513, ECO:0000303|PubMed:9537380};\n",
      "         Short=PN/PL/PM kinase {ECO:0000303|PubMed:8764513};\n",
      "         EC=2.7.1.35 {ECO:0000269|PubMed:15249053, ECO:0000269|PubMed:9537380};\n",
      "AltName: Full=B6-vitamer kinase {ECO:0000303|PubMed:8764513};\n",
      "AltName: Full=Pyridoxal kinase 1 {ECO:0000303|PubMed:15249053};\n",
      "         Short=PL kinase 1 {ECO:0000303|PubMed:15249053};\n",
      "Name=pdxK; Synonyms=yfeI; OrderedLocusNames=b2418, JW2411;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, AND DISRUPTION PHENOTYPE.\n",
      "FUNCTION, CATALYTIC ACTIVITY, DISRUPTION PHENOTYPE, AND PATHWAY.\n",
      "FUNCTION, CATALYTIC ACTIVITY, BIOPHYSICOCHEMICAL PROPERTIES, AND COFACTOR.\n",
      "-!- FUNCTION: B6-vitamer kinase involved in the salvage pathway of\n",
      "    pyridoxal 5'-phosphate (PLP). Catalyzes the phosphorylation of\n",
      "    pyridoxine (PN), pyridoxal (PL), and pyridoxamine (PM), forming their\n",
      "    respective 5'-phosphorylated esters, i.e. PNP, PLP and PMP.\n",
      "    {ECO:0000269|PubMed:15249053, ECO:0000269|PubMed:9537380}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + pyridoxal = ADP + H(+) + pyridoxal 5'-phosphate;\n",
      "      Xref=Rhea:RHEA:10224, ChEBI:CHEBI:15378, ChEBI:CHEBI:17310,\n",
      "      ChEBI:CHEBI:30616, ChEBI:CHEBI:456216, ChEBI:CHEBI:597326;\n",
      "      EC=2.7.1.35; Evidence={ECO:0000269|PubMed:15249053,\n",
      "      ECO:0000269|PubMed:9537380};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + pyridoxine = ADP + H(+) + pyridoxine 5'-phosphate;\n",
      "      Xref=Rhea:RHEA:25108, ChEBI:CHEBI:15378, ChEBI:CHEBI:16709,\n",
      "      ChEBI:CHEBI:30616, ChEBI:CHEBI:58589, ChEBI:CHEBI:456216;\n",
      "      EC=2.7.1.35; Evidence={ECO:0000269|PubMed:15249053,\n",
      "      ECO:0000269|PubMed:9537380};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + pyridoxamine = ADP + H(+) + pyridoxamine 5'-phosphate;\n",
      "      Xref=Rhea:RHEA:25104, ChEBI:CHEBI:15378, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:57761, ChEBI:CHEBI:58451, ChEBI:CHEBI:456216;\n",
      "      EC=2.7.1.35; Evidence={ECO:0000269|PubMed:15249053};\n",
      "-!- COFACTOR:\n",
      "    Name=Zn(2+); Xref=ChEBI:CHEBI:29105;\n",
      "      Evidence={ECO:0000269|PubMed:15249053};\n",
      "    Name=Mg(2+); Xref=ChEBI:CHEBI:18420;\n",
      "      Evidence={ECO:0000269|PubMed:15249053};\n",
      "    Note=Can use both zinc and magnesium that is complexed with ATP.\n",
      "    However, magnesium seems to be the preferred metal used under\n",
      "    physiological conditions. {ECO:0000269|PubMed:15249053};\n",
      "-!- ACTIVITY REGULATION: Is activated by the monovalent cation potassium.\n",
      "    {ECO:0000269|PubMed:16740960}.\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=100 uM for pyridoxal (in the presence of MgATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=50 uM for pyridoxal (in the presence of MgATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:16740960};\n",
      "      KM=190 uM for pyridoxal (in the presence of ZnATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=25 uM for pyridoxine (in the presence of MgATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=30 uM for pyridoxamine (in the presence of MgATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=10 uM for pyridoxamine (in the presence of ZnATP, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=600 uM for MgATP (at pH 7.3 and 37 degrees Celsius)\n",
      "      {ECO:0000269|PubMed:15249053};\n",
      "      KM=450 uM for MgATP (at pH 7.3 and 37 degrees Celsius)\n",
      "      {ECO:0000269|PubMed:16740960};\n",
      "      KM=2100 uM for MgATP (at pH 6.1 and 37 degrees Celsius)\n",
      "      {ECO:0000269|PubMed:15249053};\n",
      "      KM=70 uM for ZnATP (in the presence of pyridoxal, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      KM=45 uM for ZnATP (in the presence of pyridoxamine, at pH 7.3 and 37\n",
      "      degrees Celsius) {ECO:0000269|PubMed:15249053};\n",
      "      Note=kcat is 140 min(-1) for the phosphorylation of PL with MgATP.\n",
      "      kcat is 120 min(-1) for the phosphorylation of PL with ZnATP. kcat is\n",
      "      20 min(-1) for the phosphorylation of PN with MgATP. kcat is 40 min(-\n",
      "      1) for the phosphorylation of PM with MgATP. kcat is 25 min(-1) for\n",
      "      the phosphorylation of PM with ZnATP (at pH 7.3 and 37 degrees\n",
      "      Celsius) (PubMed:15249053). kcat is 250 min(-1) for the\n",
      "      phosphorylation of PL with MgATP (at pH 7.3 and 37 degrees Celsius)\n",
      "      (PubMed:16740960). {ECO:0000269|PubMed:15249053,\n",
      "      ECO:0000269|PubMed:16740960};\n",
      "-!- PATHWAY: Cofactor metabolism; pyridoxal 5'-phosphate salvage; pyridoxal\n",
      "    5'-phosphate from pyridoxal: step 1/1. {ECO:0000269|PubMed:9537380}.\n",
      "-!- PATHWAY: Cofactor metabolism; pyridoxal 5'-phosphate salvage;\n",
      "    pyridoxine 5'-phosphate from pyridoxine: step 1/1.\n",
      "    {ECO:0000269|PubMed:9537380}.\n",
      "-!- PATHWAY: Cofactor metabolism; pyridoxal 5'-phosphate salvage;\n",
      "    pyridoxamine 5'-phosphate from pyridoxamine: step 1/1.\n",
      "    {ECO:0000305|PubMed:15249053}.\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000269|PubMed:16740960}.\n",
      "-!- DISRUPTION PHENOTYPE: Cells lacking this gene lack pyridoxine kinase\n",
      "    activity but still contain pyridoxal kinase activity (PubMed:8764513).\n",
      "    Cells lacking this gene and cells lacking both pdxY and pdxK are not\n",
      "    auxotrophs, meaning that the de novo pathway of PLP biosynthesis is\n",
      "    functional. For PLP salvage, the pdxY single mutant can use both\n",
      "    pyridoxine and pyridoxal, the pdxK single mutant can use pyridoxal but\n",
      "    not pyridoxine, and the double mutant can no longer use both compounds\n",
      "    (PubMed:9537380). {ECO:0000269|PubMed:8764513,\n",
      "    ECO:0000269|PubMed:9537380}.\n",
      "-!- SIMILARITY: Belongs to the pyridoxine kinase family. PdxK subfamily.\n",
      "    {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: YKGG_ECOLI  Reviewed; 231 AA.\n",
      "ID: P77433; Q2MCB4;\n",
      "RecName: Full=Uncharacterized protein YkgG;\n",
      "Name=ykgG; OrderedLocusNames=b0308, JW5042;\n",
      "Source: Escherichia coli (strain K12).\n",
      "-!- MISCELLANEOUS: The E.coli ykgEFG operon is a clear homolog of the\n",
      "    B.subtilis lutABC operon, however it is not able of restoring L-lactate\n",
      "    utilization to the B.subtilis mutant for lutABC operon. Thus, ykgEFG\n",
      "    operon may contribute to lactate catabolism in E.coli, but under\n",
      "    conditions other than those tested, or alternatively, the ykgEFG operon\n",
      "    may be responsible for the catabolism of a metabolite other than (but\n",
      "    perhaps related to) lactate in E.coli.\n",
      "-!- SIMILARITY: Belongs to the LutC/YkgG family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAB18035.1; Type=Erroneous initiation; Note=Extended N-terminus.; Evidence={ECO:0000305};\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: RISA_ECOLI  Reviewed; 213 AA.\n",
      "ID: P0AFU8; P29015;\n",
      "RecName: Full=Riboflavin synthase;\n",
      "         Short=RS;\n",
      "         EC=2.5.1.9;\n",
      "Name=ribC; Synonyms=ribE; OrderedLocusNames=b1662, JW1654;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, CATALYTIC ACTIVITY, AND\n",
      "-!- FUNCTION: Catalyzes the dismutation of two molecules of 6,7-dimethyl-8-\n",
      "    ribityllumazine, resulting in the formation of riboflavin and 5-amino-\n",
      "    6-(D-ribitylamino)uracil. {ECO:0000269|PubMed:9022701}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=2 6,7-dimethyl-8-(1-D-ribityl)lumazine + H(+) = 5-amino-6-(D-\n",
      "      ribitylamino)uracil + riboflavin; Xref=Rhea:RHEA:20772,\n",
      "      ChEBI:CHEBI:15378, ChEBI:CHEBI:15934, ChEBI:CHEBI:57986,\n",
      "      ChEBI:CHEBI:58201; EC=2.5.1.9; Evidence={ECO:0000269|PubMed:9022701};\n",
      "-!- PATHWAY: Cofactor biosynthesis; riboflavin biosynthesis; riboflavin\n",
      "    from 2-hydroxy-3-oxobutyl phosphate and 5-amino-6-(D-\n",
      "    ribitylamino)uracil: step 2/2.\n",
      "-!- SUBUNIT: Homotrimer. Unlike in B.subtilis, does not interact with 6,7-\n",
      "    dimethyl-8-ribityllumazine synthase. {ECO:0000269|PubMed:11399071,\n",
      "    ECO:0000269|PubMed:12927541, ECO:0000269|PubMed:9022701}.\n",
      "Domínios PROSITE: PROSITE; PS51177; LUMAZINE_BIND; 2.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: SYW_ECOLI   Reviewed; 334 AA.\n",
      "ID: P00954; Q2M749;\n",
      "RecName: Full=Tryptophan--tRNA ligase {ECO:0000255|HAMAP-Rule:MF_00140};\n",
      "         EC=6.1.1.2 {ECO:0000255|HAMAP-Rule:MF_00140, ECO:0000269|PubMed:10918062};\n",
      "AltName: Full=Tryptophanyl-tRNA synthetase {ECO:0000255|HAMAP-Rule:MF_00140};\n",
      "         Short=TrpRS {ECO:0000255|HAMAP-Rule:MF_00140};\n",
      "Name=trpS {ECO:0000255|HAMAP-Rule:MF_00140};\n",
      "OrderedLocusNames=b3384, JW3347;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, CATALYTIC ACTIVITY, AND SUBSTRATE SPECIFICITY.\n",
      "-!- FUNCTION: Catalyzes the attachment of tryptophan to tRNA(Trp). Amino\n",
      "    acylates tRNA(Trp) with both L- and D-tryptophan, although D-tryptophan\n",
      "    is a poor substrate (PubMed:10918062). {ECO:0000255|HAMAP-\n",
      "    Rule:MF_00140, ECO:0000269|PubMed:10918062}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + L-tryptophan + tRNA(Trp) = AMP + diphosphate + H(+) + L-\n",
      "      tryptophanyl-tRNA(Trp); Xref=Rhea:RHEA:24080, Rhea:RHEA-COMP:9671,\n",
      "      Rhea:RHEA-COMP:9705, ChEBI:CHEBI:15378, ChEBI:CHEBI:30616,\n",
      "      ChEBI:CHEBI:33019, ChEBI:CHEBI:57912, ChEBI:CHEBI:78442,\n",
      "      ChEBI:CHEBI:78535, ChEBI:CHEBI:456215; EC=6.1.1.2;\n",
      "      Evidence={ECO:0000255|HAMAP-Rule:MF_00140,\n",
      "      ECO:0000269|PubMed:10918062};\n",
      "-!- SUBUNIT: Homodimer. {ECO:0000255|HAMAP-Rule:MF_00140}.\n",
      "Localização subcelular:  Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00140}.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00140}.\n",
      "-!- SIMILARITY: Belongs to the class-I aminoacyl-tRNA synthetase family.\n",
      "    {ECO:0000255|HAMAP-Rule:MF_00140}.\n",
      "Domínios PROSITE: PROSITE; PS00178; AA_TRNA_LIGASE_I; 1.\n"
     ]
    }
   ],
   "source": [
    "id_gene = ['P12281','P0ADG4','P02943','P06149','P32173','P32125','P0AC84','P77747','P02932','P02931','P06996','P11458','P25437','P40191','P77433','P0AFU8','P00954']\n",
    "filename = \"xanthohumol.xml\"\n",
    "file (id_gene,filename)\n",
    "informations (\"xanthohumol.xml\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2fca90f3-0508-42c9-b29f-c51bbfe75072",
   "metadata": {},
   "source": [
    "### Informations about prenylnaringenin genes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "c31f91b1-5c28-4e61-96d9-215278a36603",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: ISCU_ECOLI  Reviewed; 128 AA.\n",
      "ID: P0ACD4; P77310;\n",
      "RecName: Full=Iron-sulfur cluster assembly scaffold protein IscU;\n",
      "AltName: Full=Sulfur acceptor protein IscU;\n",
      "Name=iscU; Synonyms=nifU, yfhN; OrderedLocusNames=b2529, JW2513;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION AS A SULFUR ACCEPTOR, INTERACTION WITH ISCS, ACTIVITY REGULATION,\n",
      "FUNCTION, SUBUNIT, AND MUTAGENESIS OF ASP-39; LYS-89; ASN-90; SER-107 AND\n",
      "-!- FUNCTION: A scaffold on which IscS assembles Fe-S clusters. Exists as 2\n",
      "    interconverting forms, a structured (S) and disordered (D) form. The D-\n",
      "    state is the preferred substrate for IscS. Converts to the S-state when\n",
      "    an Fe-S cluster is assembled, which helps it dissociate from IscS to\n",
      "    transfer the Fe-S to an acceptor. It is likely that Fe-S cluster\n",
      "    coordination is flexible as the role of this complex is to build and\n",
      "    then hand off Fe-S clusters. {ECO:0000269|PubMed:11577100,\n",
      "    ECO:0000269|PubMed:22203963}.\n",
      "-!- ACTIVITY REGULATION: Carboxymethylation by iodoacetic acid blocks\n",
      "    sulfur transfer to this protein. {ECO:0000269|PubMed:11577100}.\n",
      "-!- SUBUNIT: Homodimer. Forms a heterotetramer with IscS; each subunit of\n",
      "    the IscS dimer contacts an IscU monomer. {ECO:0000269|PubMed:11577100,\n",
      "    ECO:0000269|PubMed:22203963}.\n",
      "-!- INTERACTION:\n",
      "    P0ACD4; P0A6B7: iscS; NbExp=12; IntAct=EBI-561646, EBI-550055;\n",
      "    P0ACD4; P0ACD4: iscU; NbExp=5; IntAct=EBI-561646, EBI-561646;\n",
      "-!- SIMILARITY: Belongs to the NifU family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: FOLB_ECOLI  Reviewed; 122 AA.\n",
      "ID: P0AC16; P31055; P76659; Q2M9E7;\n",
      "RecName: Full=Dihydroneopterin aldolase {ECO:0000303|PubMed:9651328};\n",
      "         Short=DHNA {ECO:0000303|PubMed:25264482};\n",
      "         EC=4.1.2.25 {ECO:0000269|PubMed:9651328};\n",
      "AltName: Full=7,8-dihydroneopterin 2'-epimerase;\n",
      "AltName: Full=7,8-dihydroneopterin aldolase;\n",
      "AltName: Full=7,8-dihydroneopterin epimerase;\n",
      "         EC=5.1.99.8 {ECO:0000269|PubMed:9651328};\n",
      "AltName: Full=Dihydroneopterin epimerase;\n",
      "Name=folB {ECO:0000303|PubMed:9651328}; Synonyms=ygiG;\n",
      "OrderedLocusNames=b3058, JW3030;\n",
      "Source: Escherichia coli (strain K12).\n",
      "FUNCTION, CATALYTIC ACTIVITY, SUBUNIT, BIOPHYSICOCHEMICAL PROPERTIES, AND\n",
      "-!- FUNCTION: Catalyzes the conversion of 7,8-dihydroneopterin to 6-\n",
      "    hydroxymethyl-7,8-dihydropterin. Can use L-threo-dihydroneopterin and\n",
      "    D-erythro-dihydroneopterin as substrates for the formation of 6-\n",
      "    hydroxymethyldihydropterin, but it can also catalyze the epimerization\n",
      "    of carbon 2' of dihydroneopterin to dihydromonapterin at appreciable\n",
      "    velocity. {ECO:0000269|PubMed:9651328}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=7,8-dihydroneopterin = 6-hydroxymethyl-7,8-dihydropterin +\n",
      "      glycolaldehyde; Xref=Rhea:RHEA:10540, ChEBI:CHEBI:17001,\n",
      "      ChEBI:CHEBI:17071, ChEBI:CHEBI:44841; EC=4.1.2.25;\n",
      "      Evidence={ECO:0000269|PubMed:9651328};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=7,8-dihydroneopterin = 7,8-dihydromonapterin;\n",
      "      Xref=Rhea:RHEA:45328, ChEBI:CHEBI:17001, ChEBI:CHEBI:71175;\n",
      "      EC=5.1.99.8; Evidence={ECO:0000269|PubMed:9651328};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=64 uM for 7,8-dihydroneopterin in aldolase reaction\n",
      "      {ECO:0000269|PubMed:9651328};\n",
      "      KM=45 uM for 7,8-dihydroneopterin in epimerase reaction\n",
      "      {ECO:0000269|PubMed:9651328};\n",
      "      KM=36 uM for 7,8-dihydromonapterin in aldolase reaction\n",
      "      {ECO:0000269|PubMed:9651328};\n",
      "      KM=57 uM for 7,8-dihydromonapterin in epimerase reaction\n",
      "      {ECO:0000269|PubMed:9651328};\n",
      "      Vmax=127 umol/h/mg enzyme toward 7,8-dihydroneopterin in aldolase\n",
      "      reaction {ECO:0000269|PubMed:9651328};\n",
      "      Vmax=20.3 umol/h/mg enzyme toward 7,8-dihydroneopterin in epimerase\n",
      "      reaction {ECO:0000269|PubMed:9651328};\n",
      "      Vmax=158 umol/h/mg enzyme toward 7,8-dihydromonapterin in aldolase\n",
      "      reaction {ECO:0000269|PubMed:9651328};\n",
      "      Vmax=15.6 umol/h/mg enzyme toward 7,8-dihydromonapterin in epimerase\n",
      "      reaction {ECO:0000269|PubMed:9651328};\n",
      "-!- PATHWAY: Cofactor biosynthesis; tetrahydrofolate biosynthesis; 2-amino-\n",
      "    4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine diphosphate from 7,8-\n",
      "    dihydroneopterin triphosphate: step 3/4.\n",
      "-!- SUBUNIT: Homooctamer. {ECO:0000269|PubMed:9651328}.\n",
      "-!- SIMILARITY: Belongs to the DHNA family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAA89138.1; Type=Erroneous initiation; Note=Extended N-terminus.; Evidence={ECO:0000305};\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: RISA_ECOLI  Reviewed; 213 AA.\n",
      "ID: P0AFU8; P29015;\n",
      "RecName: Full=Riboflavin synthase;\n",
      "         Short=RS;\n",
      "         EC=2.5.1.9;\n",
      "Name=ribC; Synonyms=ribE; OrderedLocusNames=b1662, JW1654;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, CATALYTIC ACTIVITY, AND\n",
      "-!- FUNCTION: Catalyzes the dismutation of two molecules of 6,7-dimethyl-8-\n",
      "    ribityllumazine, resulting in the formation of riboflavin and 5-amino-\n",
      "    6-(D-ribitylamino)uracil. {ECO:0000269|PubMed:9022701}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=2 6,7-dimethyl-8-(1-D-ribityl)lumazine + H(+) = 5-amino-6-(D-\n",
      "      ribitylamino)uracil + riboflavin; Xref=Rhea:RHEA:20772,\n",
      "      ChEBI:CHEBI:15378, ChEBI:CHEBI:15934, ChEBI:CHEBI:57986,\n",
      "      ChEBI:CHEBI:58201; EC=2.5.1.9; Evidence={ECO:0000269|PubMed:9022701};\n",
      "-!- PATHWAY: Cofactor biosynthesis; riboflavin biosynthesis; riboflavin\n",
      "    from 2-hydroxy-3-oxobutyl phosphate and 5-amino-6-(D-\n",
      "    ribitylamino)uracil: step 2/2.\n",
      "-!- SUBUNIT: Homotrimer. Unlike in B.subtilis, does not interact with 6,7-\n",
      "    dimethyl-8-ribityllumazine synthase. {ECO:0000269|PubMed:11399071,\n",
      "    ECO:0000269|PubMed:12927541, ECO:0000269|PubMed:9022701}.\n",
      "Domínios PROSITE: PROSITE; PS51177; LUMAZINE_BIND; 2.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: GLK_ECOLI   Reviewed; 321 AA.\n",
      "ID: P0A6V8; P46880; P78276;\n",
      "RecName: Full=Glucokinase {ECO:0000303|PubMed:9023215};\n",
      "         EC=2.7.1.2 {ECO:0000269|PubMed:9023215};\n",
      "AltName: Full=Glucose kinase;\n",
      "Name=glk {ECO:0000303|PubMed:9023215}; OrderedLocusNames=b2388, JW2385;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, CATALYTIC ACTIVITY,\n",
      "-!- FUNCTION: An ATP-dependent kinase that phosphorylates glucose to make\n",
      "    glucose-6-phosphate, not active on fructose, galactose or mannose. Not\n",
      "    highly important in wild-type E.coli as glucose is transported into the\n",
      "    cell by the PTS system as glucose 6-phosphate.\n",
      "    {ECO:0000269|PubMed:9023215}.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=ATP + D-glucose = ADP + D-glucose 6-phosphate + H(+);\n",
      "      Xref=Rhea:RHEA:17825, ChEBI:CHEBI:4167, ChEBI:CHEBI:15378,\n",
      "      ChEBI:CHEBI:30616, ChEBI:CHEBI:61548, ChEBI:CHEBI:456216; EC=2.7.1.2;\n",
      "      Evidence={ECO:0000269|PubMed:9023215};\n",
      "    PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:17826;\n",
      "      Evidence={ECO:0000269|PubMed:9023215};\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    Kinetic parameters:\n",
      "      KM=0.78 mM for glucose {ECO:0000269|PubMed:9023215};\n",
      "      KM=3.76 mM for ATP {ECO:0000269|PubMed:9023215};\n",
      "Localização subcelular:  Cytoplasm {ECO:0000269|PubMed:9023215}.\n",
      "-!- SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:9023215}.\n",
      "-!- INDUCTION: Constitutively expressed, weakly repressed by growth on\n",
      "    glucose, repression may be partially controlled by cra (fruR).\n",
      "    {ECO:0000269|PubMed:9023215}.\n",
      "-!- DISRUPTION PHENOTYPE: No glucokinase activity, no growth phenotype in\n",
      "    an otherwise wild-type cell. {ECO:0000269|PubMed:9023215}.\n",
      "-!- SIMILARITY: Belongs to the bacterial glucokinase family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: LTAE_ECOLI  Reviewed; 333 AA.\n",
      "ID: P75823;\n",
      "RecName: Full=Low specificity L-threonine aldolase;\n",
      "         Short=Low specificity L-TA;\n",
      "         EC=4.1.2.48;\n",
      "Name=ltaE; Synonyms=ybjU; OrderedLocusNames=b0870, JW0854;\n",
      "Source: Escherichia coli (strain K12).\n",
      "-!- FUNCTION: Catalyzes the cleavage of L-allo-threonine and L-threonine to\n",
      "    glycine and acetaldehyde. L-threo-phenylserine and L-erythro-\n",
      "    phenylserine are also good substrates.\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=L-threonine = acetaldehyde + glycine; Xref=Rhea:RHEA:19625,\n",
      "      ChEBI:CHEBI:15343, ChEBI:CHEBI:57305, ChEBI:CHEBI:57926; EC=4.1.2.48;\n",
      "      Evidence={ECO:0000269|PubMed:9692922};\n",
      "-!- CATALYTIC ACTIVITY:\n",
      "    Reaction=L-allo-threonine = acetaldehyde + glycine;\n",
      "      Xref=Rhea:RHEA:26209, ChEBI:CHEBI:15343, ChEBI:CHEBI:57305,\n",
      "      ChEBI:CHEBI:58585; EC=4.1.2.48;\n",
      "      Evidence={ECO:0000269|PubMed:9692922};\n",
      "-!- COFACTOR:\n",
      "    Name=pyridoxal 5'-phosphate; Xref=ChEBI:CHEBI:597326;\n",
      "-!- BIOPHYSICOCHEMICAL PROPERTIES:\n",
      "    pH dependence:\n",
      "      Optimum pH is 8.5-9.0.;\n",
      "    Temperature dependence:\n",
      "      Optimum temperature is 65-70 degrees Celsius.;\n",
      "-!- SUBUNIT: Homotetramer. {ECO:0000305}.\n",
      "-!- INTERACTION:\n",
      "    P75823; P75823: ltaE; NbExp=2; IntAct=EBI-1120660, EBI-1120660;\n",
      "-!- SIMILARITY: Belongs to the threonine aldolase family. {ECO:0000305}.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: DPPA_ECOLI  Reviewed; 535 AA.\n",
      "ID: P23847; Q2M7K6;\n",
      "RecName: Full=Dipeptide-binding protein {ECO:0000303|PubMed:1702779, ECO:0000303|PubMed:1956284};\n",
      "         Short=DBP {ECO:0000303|PubMed:1956284};\n",
      "AltName: Full=Periplasmic dipeptide transport protein {ECO:0000303|PubMed:1702779};\n",
      "Flags: Precursor;\n",
      "Name=dppA {ECO:0000303|PubMed:1702779, ECO:0000303|PubMed:1956284};\n",
      "OrderedLocusNames=b3544, JW3513;\n",
      "Source: Escherichia coli (strain K12).\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, SUBCELLULAR LOCATION,\n",
      "NUCLEOTIDE SEQUENCE [GENOMIC DNA], FUNCTION, AND SUBUNIT.\n",
      "PROTEIN SEQUENCE OF 29-61, FUNCTION, BINDING SPECIFICITY, AND DOMAIN.\n",
      "FUNCTION.\n",
      "FUNCTION IN 5-AMINOLEVULINIC ACID TRANSPORT.\n",
      "FUNCTION IN HEME TRANSPORT, ACTIVITY REGULATION, AND DISRUPTION PHENOTYPE.\n",
      "GLY-LEU, FUNCTION, AND DOMAIN.\n",
      "-!- FUNCTION: Part of the ABC transporter DppABCDF involved in dipeptide\n",
      "    transport (PubMed:1702779, PubMed:7536291, PubMed:10537211). Binds\n",
      "    dipeptides and accepts a wide range of side chains, including small\n",
      "    neutral, bulky hydrophobic, and positively and negatively charged\n",
      "    groups (PubMed:10537211). Tripeptides are poor substrates\n",
      "    (PubMed:10537211). DppA alone controls the specificity of the Dpp\n",
      "    transporter (PubMed:10537211). In addition, plays a role in chemotaxis\n",
      "    toward peptides via interaction with the chemotaxis protein Tap\n",
      "    (PubMed:3520334, PubMed:8563629). {ECO:0000269|PubMed:10537211,\n",
      "    ECO:0000269|PubMed:1702779, ECO:0000269|PubMed:3520334,\n",
      "    ECO:0000269|PubMed:7536291, ECO:0000269|PubMed:8563629}.\n",
      "-!- FUNCTION: Binds heme. When a foreign outer membrane heme receptor is\n",
      "    expressed in E.coli, DppABCDF can also transport heme and its\n",
      "    precursor, 5-aminolevulinic acid (ALA), from the periplasm into the\n",
      "    cytoplasm. {ECO:0000269|PubMed:16905647, ECO:0000269|PubMed:8444807}.\n",
      "-!- ACTIVITY REGULATION: Heme binding is inhibited by dipeptide.\n",
      "    {ECO:0000269|PubMed:16905647}.\n",
      "-!- SUBUNIT: The complex is composed of two ATP-binding proteins (DppD and\n",
      "    DppF), two transmembrane proteins (DppB and DppC) and a solute-binding\n",
      "    protein (DppA). {ECO:0000269|PubMed:7536291}.\n",
      "Localização subcelular:  Periplasm {ECO:0000269|PubMed:1702779}.\n",
      "-!- SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:1702779}.\n",
      "-!- INDUCTION: Expression is repressed by the presence of casamino acids.\n",
      "    {ECO:0000269|PubMed:1702779}.\n",
      "-!- DOMAIN: Consists of two distinct domains (I and II) connected by two\n",
      "    strands that presumably function as a hinge (PubMed:8527431,\n",
      "    PubMed:8563629). Undergoes conformational change upon substrate binding\n",
      "    (PubMed:10537211). {ECO:0000269|PubMed:10537211,\n",
      "    ECO:0000269|PubMed:8527431, ECO:0000269|PubMed:8563629}.\n",
      "-!- DISRUPTION PHENOTYPE: Inactivation of the gene results in the inability\n",
      "    of a proline auxotroph to utilize Pro-Gly as a proline source\n",
      "    (PubMed:1702779). Inactivation of the gene has no effect on iron-heme\n",
      "    utilization, but the double mppA dppA mutant is unable to use heme as\n",
      "    iron source (PubMed:16905647). {ECO:0000269|PubMed:16905647,\n",
      "    ECO:0000269|PubMed:1702779}.\n",
      "-!- SIMILARITY: Belongs to the bacterial solute-binding protein 5 family.\n",
      "    {ECO:0000305}.\n",
      "Domínios PROSITE: PROSITE; PS01040; SBP_BACTERIAL_5; 1.\n",
      "---------------------------\n",
      "\n",
      "Nome de acesso, Status e comprimento da sequência: CCMC_ECOLI  Reviewed; 245 AA.\n",
      "ID: P0ABM1; P33929; Q2MAP4;\n",
      "RecName: Full=Heme exporter protein C;\n",
      "AltName: Full=Cytochrome c-type biogenesis protein CcmC;\n",
      "Name=ccmC; Synonyms=yejT, yejU; OrderedLocusNames=b2199, JW2187;\n",
      "Source: Escherichia coli (strain K12).\n",
      "-!- FUNCTION: Required for the export of heme to the periplasm for the\n",
      "    biogenesis of c-type cytochromes.\n",
      "-!- INTERACTION:\n",
      "    P0ABM1; P33931: ccmA; NbExp=4; IntAct=EBI-2123469, EBI-6402796;\n",
      "    P0ABM1; P0ABM5: ccmD; NbExp=4; IntAct=EBI-2123469, EBI-3894922;\n",
      "    P0ABM1; P69490: ccmE; NbExp=7; IntAct=EBI-2123469, EBI-1128007;\n",
      "Localização subcelular:  Cell inner membrane; Multi-pass membrane protein.\n",
      "-!- SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membrane protein.\n",
      "-!- SIMILARITY: Belongs to the CcmC/CycZ/HelC family. {ECO:0000305}.\n",
      "-!- SEQUENCE CAUTION:\n",
      "    Sequence=AAA16391.1; Type=Frameshift; Evidence={ECO:0000305};\n"
     ]
    }
   ],
   "source": [
    "id_gene = ['P0ACD4','P0AC16','P0AFU8','P0A6V8','P75823','P23847','P0ABM1']\n",
    "filename = \"prenylnaringenin.xml\"\n",
    "file (id_gene,filename)\n",
    "informations (\"prenylnaringenin.xml\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
